  INFO root [2018-05-21 08:50:00,810] - 
  INFO root [2018-05-21 08:50:00,810] - -----------------------------------------------------------------------------------------


  INFO root [2018-05-21 08:50:00,811] - Resmgr::initialize: KBE_ROOT=D:/noah/ChessWarrior/server/
  INFO root [2018-05-21 08:50:00,811] - Resmgr::initialize: KBE_RES_PATH=D:/noah/ChessWarrior/server/kbe/res/;D:/noah/ChessWarrior/server/chess_assets/;D:/noah/ChessWarrior/server/chess_assets/scripts/;D:/noah/ChessWarrior/server/chess_assets/res/
  INFO root [2018-05-21 08:50:00,811] - Resmgr::initialize: KBE_BIN_PATH=D:/noah/ChessWarrior/server/kbe/bin/server/
  INFO root [2018-05-21 08:50:00,814] - NetworkInterface::initialize(EXTERNAL): bound to all interfaces with default route interface on 192.168.0.111 ( 192.168.0.111:20013 )
  INFO root [2018-05-21 08:50:00,814] - NetworkInterface::initialize(EXTERNAL): address 192.168.0.111:20013, SOMAXCONN=128.
  INFO root [2018-05-21 08:50:00,815] - NetworkInterface::initialize(INTERNAL): bound to all interfaces with default route interface on 192.168.0.111 ( 192.168.0.111:52868 )
  INFO root [2018-05-21 08:50:00,815] - NetworkInterface::initialize(INTERNAL): address 192.168.0.111:52868, SOMAXCONN=128.
  INFO root [2018-05-21 08:50:00,815] - server-configs:
  INFO root [2018-05-21 08:50:00,815] - 	internalAddr : 192.168.0.111:52868
  INFO root [2018-05-21 08:50:00,815] - 	externalAddr : 192.168.0.111:20013
  INFO root [2018-05-21 08:50:00,815] - 	componentID : 9000
  INFO root [2018-05-21 08:50:01,005] - Components::findLogger: found logger, addr:192.168.0.111:52776
  INFO root [2018-05-21 08:50:01,005] - Components::addComponent: reset orderLog, uid=17358!
  INFO root [2018-05-21 08:50:01,006] - Components::addComponent[logger], uid=17358, componentID=2000, globalorderid=1, grouporderid=1, totalcount=1
  INFO root [2018-05-21 08:50:01,011] - ---- loginapp Version: 2.0.0. ScriptVersion: 0.1.0. Protocol: B20F257E332779D9E1EA7D34638E6146. Config: Debug X86. Built: 15:50:13 May 15 2018. AppID: 9000. UID: 17358. PID: 8316 ----
  INFO root [2018-05-21 08:50:01,011] - Server : Intel(R) Core(TM) i5-7500 CPU @ 3.40GHz with 2097151kB RAM
  INFO root [2018-05-21 08:50:01,012] - ThreadPool::createThreadPool: creating  threadpool...
  INFO root [2018-05-21 08:50:01,013] - ThreadPool::createThreadPool: successfully(2), newThreadCount=1, normalMaxThreadCount=2, maxThreadCount=8
 DEBUG root [2018-05-21 08:50:01,142] - Script::install_py_dlls(): PyInit_pyexpat
 DEBUG root [2018-05-21 08:50:01,142] - Script::install_py_dlls(): PyInit__socket
 DEBUG root [2018-05-21 08:50:01,143] - Script::install_py_dlls(): PyInit__ssl
 DEBUG root [2018-05-21 08:50:01,145] - Script::install_py_dlls(): PyInit__hashlib
 DEBUG root [2018-05-21 08:50:01,146] - Script::install_py_dlls(): PyInit_select
 DEBUG root [2018-05-21 08:50:01,146] - Script::install_py_dlls(): PyInit__ctypes
 DEBUG root [2018-05-21 08:50:01,146] - Script::install_py_dlls(): PyInit__elementtree
 DEBUG root [2018-05-21 08:50:01,185] - Script::install_py_dlls(): PyInit_unicodedata
  INFO root [2018-05-21 08:50:01,220] - Script::install(): is successfully, Python=(3.4.2 (default, May 15 2018, 15:50:12) [MSC v.1900 32 bit (Intel)])!
  INFO root [2018-05-21 08:50:01,220] - Script::install(): KBExtra is successfully!
S_INFO root [2018-05-21 08:50:01,226] - onLoginAppReady: bootstrapGroupIndex=0, bootstrapGlobalIndex=0
  INFO root [2018-05-21 08:50:01,226] - TelnetServer server is running on port 31000
  INFO root [2018-05-21 08:50:01,599] - ---- loginapp is running ----
 DEBUG root [2018-05-21 08:50:01,599] - Components::process(): Request for the process of identity...
  WARN root [2018-05-21 08:50:07,730] - Forwarding logs to logger[192.168.0.111:52776/0/0/0]...
 ERROR root [2018-05-21 08:50:59,056] - Components::removeComponentByChannel: logger : 2000, Abnormal exit! errorno=10054
  INFO root [2018-05-21 08:50:59,066] - The following logs sent to logger failed:
  WARN root [2018-05-21 08:50:59,117] - ==> [2018-05-21 08:50:59 053] TCPPacketReceiver::processPendingEvents(192.168.0.111:52776): Throwing REASON_GENERAL_NETWORK - WSAECONNRESET
 ERROR root [2018-05-21 08:50:59,118] - ==> [2018-05-21 08:50:59 057] Components::removeComponentByChannel: logger : 2000, Abnormal exit! errorno=10054
  WARN root [2018-05-21 08:50:59,118] - TCPPacketReceiver::processPendingEvents(192.168.0.111:52797): Throwing REASON_GENERAL_NETWORK - WSAECONNRESET
 ERROR root [2018-05-21 08:50:59,119] - Components::removeComponentByChannel: dbmgr : 4000, Abnormal exit! errorno=10054
  INFO root [2018-05-21 08:50:59,119] - Shutdowner::onShutdownBegin: shutting down(period=0, tickPeriod=1)
S_INFO root [2018-05-21 08:50:59,130] - onLoginAppShutDown()
  INFO root [2018-05-21 08:50:59,130] - Shutdowner::onShutdown: shutting down
  INFO root [2018-05-21 08:50:59,130] - Shutdowner::onShutdownEnd: shutting down
 DEBUG root [2018-05-21 08:50:59,152] - Script::uninstall_py_dlls(): PyInit_pyexpat
 DEBUG root [2018-05-21 08:50:59,153] - Script::uninstall_py_dlls(): PyInit__socket
 DEBUG root [2018-05-21 08:50:59,153] - Script::uninstall_py_dlls(): PyInit__ssl
 DEBUG root [2018-05-21 08:50:59,153] - Script::uninstall_py_dlls(): PyInit__hashlib
 DEBUG root [2018-05-21 08:50:59,153] - Script::uninstall_py_dlls(): PyInit_select
 DEBUG root [2018-05-21 08:50:59,153] - Script::uninstall_py_dlls(): PyInit__ctypes
 DEBUG root [2018-05-21 08:50:59,153] - Script::uninstall_py_dlls(): PyInit__elementtree
 DEBUG root [2018-05-21 08:50:59,153] - Script::uninstall_py_dlls(): PyInit_unicodedata
  INFO root [2018-05-21 08:50:59,167] - Script::uninstall(): is successfully!
 DEBUG root [2018-05-21 08:50:59,192] - ThreadPool::destroy(): starting size 2.
  INFO root [2018-05-21 08:51:02,908] - 
  INFO root [2018-05-21 08:51:02,909] - -----------------------------------------------------------------------------------------


  INFO root [2018-05-21 08:51:02,909] - Resmgr::initialize: KBE_ROOT=D:/noah/ChessWarrior/server/
  INFO root [2018-05-21 08:51:02,910] - Resmgr::initialize: KBE_RES_PATH=D:/noah/ChessWarrior/server/kbe/res/;D:/noah/ChessWarrior/server/chess_assets/;D:/noah/ChessWarrior/server/chess_assets/scripts/;D:/noah/ChessWarrior/server/chess_assets/res/
  INFO root [2018-05-21 08:51:02,910] - Resmgr::initialize: KBE_BIN_PATH=D:/noah/ChessWarrior/server/kbe/bin/server/
  INFO root [2018-05-21 08:51:02,912] - NetworkInterface::initialize(EXTERNAL): bound to all interfaces with default route interface on 192.168.0.111 ( 192.168.0.111:20013 )
  INFO root [2018-05-21 08:51:02,912] - NetworkInterface::initialize(EXTERNAL): address 192.168.0.111:20013, SOMAXCONN=128.
  INFO root [2018-05-21 08:51:02,913] - NetworkInterface::initialize(INTERNAL): bound to all interfaces with default route interface on 192.168.0.111 ( 192.168.0.111:52956 )
  INFO root [2018-05-21 08:51:02,913] - NetworkInterface::initialize(INTERNAL): address 192.168.0.111:52956, SOMAXCONN=128.
  INFO root [2018-05-21 08:51:02,913] - server-configs:
  INFO root [2018-05-21 08:51:02,913] - 	internalAddr : 192.168.0.111:52956
  INFO root [2018-05-21 08:51:02,913] - 	externalAddr : 192.168.0.111:20013
  INFO root [2018-05-21 08:51:02,913] - 	componentID : 9000
  INFO root [2018-05-21 08:51:04,565] - ---- loginapp Version: 2.0.0. ScriptVersion: 0.1.0. Protocol: B20F257E332779D9E1EA7D34638E6146. Config: Debug X86. Built: 15:50:13 May 15 2018. AppID: 9000. UID: 17756. PID: 14632 ----
  INFO root [2018-05-21 08:51:04,586] - Server : Intel(R) Core(TM) i5-7500 CPU @ 3.40GHz with 2097151kB RAM
  INFO root [2018-05-21 08:51:04,591] - ThreadPool::createThreadPool: creating  threadpool...
  INFO root [2018-05-21 08:51:04,591] - ThreadPool::createThreadPool: successfully(2), newThreadCount=1, normalMaxThreadCount=2, maxThreadCount=8
 DEBUG root [2018-05-21 08:51:04,721] - Script::install_py_dlls(): PyInit_pyexpat
 DEBUG root [2018-05-21 08:51:04,722] - Script::install_py_dlls(): PyInit__socket
 DEBUG root [2018-05-21 08:51:04,722] - Script::install_py_dlls(): PyInit__ssl
 DEBUG root [2018-05-21 08:51:04,726] - Script::install_py_dlls(): PyInit__hashlib
 DEBUG root [2018-05-21 08:51:04,727] - Script::install_py_dlls(): PyInit_select
 DEBUG root [2018-05-21 08:51:04,727] - Script::install_py_dlls(): PyInit__ctypes
 DEBUG root [2018-05-21 08:51:04,728] - Script::install_py_dlls(): PyInit__elementtree
 DEBUG root [2018-05-21 08:51:04,769] - Script::install_py_dlls(): PyInit_unicodedata
  INFO root [2018-05-21 08:51:04,809] - Script::install(): is successfully, Python=(3.4.2 (default, May 15 2018, 15:50:12) [MSC v.1900 32 bit (Intel)])!
  INFO root [2018-05-21 08:51:04,809] - Script::install(): KBExtra is successfully!
S_INFO root [2018-05-21 08:51:04,812] - onLoginAppReady: bootstrapGroupIndex=0, bootstrapGlobalIndex=0
  INFO root [2018-05-21 08:51:04,812] - TelnetServer server is running on port 31000
  INFO root [2018-05-21 08:51:05,198] - ---- loginapp is running ----
 DEBUG root [2018-05-21 08:51:05,198] - Components::process(): Request for the process of identity...
  INFO root [2018-05-21 08:51:12,332] - Components::findComponents: find logger(1)...
  INFO root [2018-05-21 08:51:12,423] - Components::findComponents: found logger, addr:192.168.0.111:52954
  INFO root [2018-05-21 08:51:12,423] - Components::addComponent: reset orderLog, uid=17756!
  INFO root [2018-05-21 08:51:12,423] - Components::addComponent[logger], uid=17756, componentID=2000, globalorderid=1, grouporderid=1, totalcount=1
  INFO root [2018-05-21 08:51:12,424] - Components::findComponents: find dbmgr(2)...
  INFO root [2018-05-21 08:51:12,528] - Components::findComponents: found dbmgr, addr:192.168.0.111:52955
  INFO root [2018-05-21 08:51:12,529] - Components::addComponent: reset orderLog, uid=17756!
  INFO root [2018-05-21 08:51:12,529] - Components::addComponent[dbmgr], uid=17756, componentID=4000, globalorderid=1, grouporderid=1, totalcount=1
  INFO root [2018-05-21 08:51:12,529] - Components::findComponents: found dbmgr, addr:192.168.0.111:52955
  WARN root [2018-05-21 08:51:13,530] - Forwarding logs to logger[192.168.0.111:52954/0/0/0]...
 ERROR root [2018-05-21 08:51:56,786] - Components::removeComponentByChannel: logger : 2000, Abnormal exit! errorno=10054
  INFO root [2018-05-21 08:51:56,790] - The following logs sent to logger failed:
  WARN root [2018-05-21 08:51:56,790] - ==> [2018-05-21 08:51:56 785] TCPPacketReceiver::processPendingEvents(192.168.0.111:52954): Throwing REASON_GENERAL_NETWORK - WSAECONNRESET
 ERROR root [2018-05-21 08:51:56,790] - ==> [2018-05-21 08:51:56 787] Components::removeComponentByChannel: logger : 2000, Abnormal exit! errorno=10054
  WARN root [2018-05-21 08:51:56,831] - TCPPacketReceiver::processPendingEvents(192.168.0.111:52955): Throwing REASON_GENERAL_NETWORK - WSAECONNRESET
 ERROR root [2018-05-21 08:51:56,832] - Components::removeComponentByChannel: dbmgr : 4000, Abnormal exit! errorno=10054
  INFO root [2018-05-21 08:51:56,835] - Shutdowner::onShutdownBegin: shutting down(period=0, tickPeriod=1)
S_INFO root [2018-05-21 08:51:56,835] - onLoginAppShutDown()
  INFO root [2018-05-21 08:51:56,835] - Shutdowner::onShutdown: shutting down
  INFO root [2018-05-21 08:51:56,835] - Shutdowner::onShutdownEnd: shutting down
 DEBUG root [2018-05-21 08:51:56,836] - Script::uninstall_py_dlls(): PyInit_pyexpat
 DEBUG root [2018-05-21 08:51:56,836] - Script::uninstall_py_dlls(): PyInit__socket
 DEBUG root [2018-05-21 08:51:56,836] - Script::uninstall_py_dlls(): PyInit__ssl
 DEBUG root [2018-05-21 08:51:56,836] - Script::uninstall_py_dlls(): PyInit__hashlib
 DEBUG root [2018-05-21 08:51:56,836] - Script::uninstall_py_dlls(): PyInit_select
 DEBUG root [2018-05-21 08:51:56,836] - Script::uninstall_py_dlls(): PyInit__ctypes
 DEBUG root [2018-05-21 08:51:56,836] - Script::uninstall_py_dlls(): PyInit__elementtree
 DEBUG root [2018-05-21 08:51:56,836] - Script::uninstall_py_dlls(): PyInit_unicodedata
  INFO root [2018-05-21 08:51:56,848] - Script::uninstall(): is successfully!
 DEBUG root [2018-05-21 08:51:56,848] - ThreadPool::destroy(): starting size 2.
  WARN root [2018-05-21 08:51:57,149] - ThreadPool::destroy(): waiting for thread(2)[0x46b6878,0x46b6e90,], try=1
  INFO root [2018-05-21 08:52:00,184] - 
  INFO root [2018-05-21 08:52:00,194] - -----------------------------------------------------------------------------------------


  INFO root [2018-05-21 08:52:00,194] - Resmgr::initialize: KBE_ROOT=D:/noah/ChessWarrior/server/
  INFO root [2018-05-21 08:52:00,195] - Resmgr::initialize: KBE_RES_PATH=D:/noah/ChessWarrior/server/kbe/res/;D:/noah/ChessWarrior/server/chess_assets/;D:/noah/ChessWarrior/server/chess_assets/scripts/;D:/noah/ChessWarrior/server/chess_assets/res/
  INFO root [2018-05-21 08:52:00,195] - Resmgr::initialize: KBE_BIN_PATH=D:/noah/ChessWarrior/server/kbe/bin/server/
  INFO root [2018-05-21 08:52:00,228] - NetworkInterface::initialize(EXTERNAL): bound to all interfaces with default route interface on 192.168.0.111 ( 192.168.0.111:20013 )
  INFO root [2018-05-21 08:52:00,228] - NetworkInterface::initialize(EXTERNAL): address 192.168.0.111:20013, SOMAXCONN=128.
  INFO root [2018-05-21 08:52:00,234] - NetworkInterface::initialize(INTERNAL): bound to all interfaces with default route interface on 192.168.0.111 ( 192.168.0.111:53075 )
  INFO root [2018-05-21 08:52:00,234] - NetworkInterface::initialize(INTERNAL): address 192.168.0.111:53075, SOMAXCONN=128.
  INFO root [2018-05-21 08:52:00,234] - server-configs:
  INFO root [2018-05-21 08:52:00,234] - 	internalAddr : 192.168.0.111:53075
  INFO root [2018-05-21 08:52:00,234] - 	externalAddr : 192.168.0.111:20013
  INFO root [2018-05-21 08:52:00,235] - 	componentID : 9000
  INFO root [2018-05-21 08:52:01,903] - ---- loginapp Version: 2.0.0. ScriptVersion: 0.1.0. Protocol: B20F257E332779D9E1EA7D34638E6146. Config: Debug X86. Built: 15:50:13 May 15 2018. AppID: 9000. UID: 17942. PID: 14580 ----
  INFO root [2018-05-21 08:52:01,911] - Server : Intel(R) Core(TM) i5-7500 CPU @ 3.40GHz with 2097151kB RAM
  INFO root [2018-05-21 08:52:01,919] - ThreadPool::createThreadPool: creating  threadpool...
  INFO root [2018-05-21 08:52:01,919] - ThreadPool::createThreadPool: successfully(2), newThreadCount=1, normalMaxThreadCount=2, maxThreadCount=8
 DEBUG root [2018-05-21 08:52:02,136] - Script::install_py_dlls(): PyInit_pyexpat
 DEBUG root [2018-05-21 08:52:02,136] - Script::install_py_dlls(): PyInit__socket
 DEBUG root [2018-05-21 08:52:02,137] - Script::install_py_dlls(): PyInit__ssl
 DEBUG root [2018-05-21 08:52:02,140] - Script::install_py_dlls(): PyInit__hashlib
 DEBUG root [2018-05-21 08:52:02,140] - Script::install_py_dlls(): PyInit_select
 DEBUG root [2018-05-21 08:52:02,140] - Script::install_py_dlls(): PyInit__ctypes
 DEBUG root [2018-05-21 08:52:02,141] - Script::install_py_dlls(): PyInit__elementtree
 DEBUG root [2018-05-21 08:52:02,205] - Script::install_py_dlls(): PyInit_unicodedata
  INFO root [2018-05-21 08:52:02,250] - Script::install(): is successfully, Python=(3.4.2 (default, May 15 2018, 15:50:12) [MSC v.1900 32 bit (Intel)])!
  INFO root [2018-05-21 08:52:02,255] - Script::install(): KBExtra is successfully!
S_INFO root [2018-05-21 08:52:02,259] - onLoginAppReady: bootstrapGroupIndex=0, bootstrapGlobalIndex=0
  INFO root [2018-05-21 08:52:02,260] - TelnetServer server is running on port 31000
  INFO root [2018-05-21 08:52:02,654] - ---- loginapp is running ----
 DEBUG root [2018-05-21 08:52:02,654] - Components::process(): Request for the process of identity...
  INFO root [2018-05-21 08:52:09,799] - Components::findComponents: find logger(1)...
  INFO root [2018-05-21 08:52:09,815] - Components::findComponents: found logger, addr:192.168.0.111:53069
  INFO root [2018-05-21 08:52:09,815] - Components::addComponent: reset orderLog, uid=17942!
  INFO root [2018-05-21 08:52:09,815] - Components::addComponent[logger], uid=17942, componentID=2000, globalorderid=1, grouporderid=1, totalcount=1
  INFO root [2018-05-21 08:52:09,817] - Components::findComponents: find dbmgr(2)...
  INFO root [2018-05-21 08:52:09,921] - Components::findComponents: found dbmgr, addr:192.168.0.111:53073
  INFO root [2018-05-21 08:52:09,921] - Components::addComponent: reset orderLog, uid=17942!
  INFO root [2018-05-21 08:52:09,922] - Components::addComponent[dbmgr], uid=17942, componentID=4000, globalorderid=1, grouporderid=1, totalcount=1
  INFO root [2018-05-21 08:52:09,922] - Components::findComponents: found dbmgr, addr:192.168.0.111:53073
  WARN root [2018-05-21 08:52:10,922] - Forwarding logs to logger[192.168.0.111:53069/0/0/0]...
 ERROR root [2018-05-21 09:03:18,003] - Components::removeComponentByChannel: logger : 2000, Abnormal exit! errorno=10054
  INFO root [2018-05-21 09:03:18,007] - The following logs sent to logger failed:
  WARN root [2018-05-21 09:03:18,007] - ==> [2018-05-21 09:03:18 000] TCPPacketReceiver::processPendingEvents(192.168.0.111:53069): Throwing REASON_GENERAL_NETWORK - WSAECONNRESET
 ERROR root [2018-05-21 09:03:18,007] - ==> [2018-05-21 09:03:18 003] Components::removeComponentByChannel: logger : 2000, Abnormal exit! errorno=10054
  WARN root [2018-05-21 09:03:18,047] - TCPPacketReceiver::processPendingEvents(192.168.0.111:53073): Throwing REASON_GENERAL_NETWORK - WSAECONNRESET
 ERROR root [2018-05-21 09:03:18,053] - Components::removeComponentByChannel: dbmgr : 4000, Abnormal exit! errorno=10054
  INFO root [2018-05-21 09:03:18,061] - Shutdowner::onShutdownBegin: shutting down(period=0, tickPeriod=1)
S_INFO root [2018-05-21 09:03:18,062] - onLoginAppShutDown()
  INFO root [2018-05-21 09:03:18,063] - Shutdowner::onShutdown: shutting down
  INFO root [2018-05-21 09:03:18,063] - Shutdowner::onShutdownEnd: shutting down
 DEBUG root [2018-05-21 09:03:18,063] - Script::uninstall_py_dlls(): PyInit_pyexpat
 DEBUG root [2018-05-21 09:03:18,063] - Script::uninstall_py_dlls(): PyInit__socket
 DEBUG root [2018-05-21 09:03:18,063] - Script::uninstall_py_dlls(): PyInit__ssl
 DEBUG root [2018-05-21 09:03:18,063] - Script::uninstall_py_dlls(): PyInit__hashlib
 DEBUG root [2018-05-21 09:03:18,063] - Script::uninstall_py_dlls(): PyInit_select
 DEBUG root [2018-05-21 09:03:18,063] - Script::uninstall_py_dlls(): PyInit__ctypes
 DEBUG root [2018-05-21 09:03:18,063] - Script::uninstall_py_dlls(): PyInit__elementtree
 DEBUG root [2018-05-21 09:03:18,064] - Script::uninstall_py_dlls(): PyInit_unicodedata
  INFO root [2018-05-21 09:03:18,075] - Script::uninstall(): is successfully!
 DEBUG root [2018-05-21 09:03:18,075] - ThreadPool::destroy(): starting size 2.
  WARN root [2018-05-21 09:03:18,376] - ThreadPool::destroy(): waiting for thread(2)[0x73aa08,0x73ace0,], try=1
  INFO root [2018-05-21 09:03:27,566] - 
  INFO root [2018-05-21 09:03:27,567] - -----------------------------------------------------------------------------------------


  INFO root [2018-05-21 09:03:27,568] - Resmgr::initialize: KBE_ROOT=D:/noah/ChessWarrior/server/
  INFO root [2018-05-21 09:03:27,568] - Resmgr::initialize: KBE_RES_PATH=D:/noah/ChessWarrior/server/kbe/res/;D:/noah/ChessWarrior/server/chess_assets/;D:/noah/ChessWarrior/server/chess_assets/scripts/;D:/noah/ChessWarrior/server/chess_assets/res/
  INFO root [2018-05-21 09:03:27,568] - Resmgr::initialize: KBE_BIN_PATH=D:/noah/ChessWarrior/server/kbe/bin/server/
  INFO root [2018-05-21 09:03:27,570] - NetworkInterface::initialize(EXTERNAL): bound to all interfaces with default route interface on 192.168.0.111 ( 192.168.0.111:20013 )
  INFO root [2018-05-21 09:03:27,570] - NetworkInterface::initialize(EXTERNAL): address 192.168.0.111:20013, SOMAXCONN=128.
  INFO root [2018-05-21 09:03:27,572] - NetworkInterface::initialize(INTERNAL): bound to all interfaces with default route interface on 192.168.0.111 ( 192.168.0.111:54031 )
  INFO root [2018-05-21 09:03:27,572] - NetworkInterface::initialize(INTERNAL): address 192.168.0.111:54031, SOMAXCONN=128.
  INFO root [2018-05-21 09:03:27,572] - server-configs:
  INFO root [2018-05-21 09:03:27,572] - 	internalAddr : 192.168.0.111:54031
  INFO root [2018-05-21 09:03:27,572] - 	externalAddr : 192.168.0.111:20013
  INFO root [2018-05-21 09:03:27,572] - 	componentID : 9000
  INFO root [2018-05-21 09:03:29,217] - ---- loginapp Version: 2.0.0. ScriptVersion: 0.1.0. Protocol: B20F257E332779D9E1EA7D34638E6146. Config: Debug X86. Built: 15:50:13 May 15 2018. AppID: 9000. UID: 20189. PID: 17916 ----
  INFO root [2018-05-21 09:03:29,218] - Server : Intel(R) Core(TM) i5-7500 CPU @ 3.40GHz with 2097151kB RAM
  INFO root [2018-05-21 09:03:29,218] - ThreadPool::createThreadPool: creating  threadpool...
  INFO root [2018-05-21 09:03:29,218] - ThreadPool::createThreadPool: successfully(2), newThreadCount=1, normalMaxThreadCount=2, maxThreadCount=8
 DEBUG root [2018-05-21 09:03:29,352] - Script::install_py_dlls(): PyInit_pyexpat
 DEBUG root [2018-05-21 09:03:29,352] - Script::install_py_dlls(): PyInit__socket
 DEBUG root [2018-05-21 09:03:29,353] - Script::install_py_dlls(): PyInit__ssl
 DEBUG root [2018-05-21 09:03:29,355] - Script::install_py_dlls(): PyInit__hashlib
 DEBUG root [2018-05-21 09:03:29,355] - Script::install_py_dlls(): PyInit_select
 DEBUG root [2018-05-21 09:03:29,356] - Script::install_py_dlls(): PyInit__ctypes
 DEBUG root [2018-05-21 09:03:29,356] - Script::install_py_dlls(): PyInit__elementtree
 DEBUG root [2018-05-21 09:03:29,397] - Script::install_py_dlls(): PyInit_unicodedata
  INFO root [2018-05-21 09:03:29,439] - Script::install(): is successfully, Python=(3.4.2 (default, May 15 2018, 15:50:12) [MSC v.1900 32 bit (Intel)])!
  INFO root [2018-05-21 09:03:29,439] - Script::install(): KBExtra is successfully!
S_INFO root [2018-05-21 09:03:29,441] - onLoginAppReady: bootstrapGroupIndex=0, bootstrapGlobalIndex=0
  INFO root [2018-05-21 09:03:29,442] - TelnetServer server is running on port 31000
  INFO root [2018-05-21 09:03:29,925] - ---- loginapp is running ----
 DEBUG root [2018-05-21 09:03:29,925] - Components::process(): Request for the process of identity...
  INFO root [2018-05-21 09:03:37,069] - Components::findComponents: find logger(1)...
  INFO root [2018-05-21 09:03:37,149] - Components::findComponents: found logger, addr:192.168.0.111:54037
  INFO root [2018-05-21 09:03:37,149] - Components::addComponent: reset orderLog, uid=20189!
  INFO root [2018-05-21 09:03:37,149] - Components::addComponent[logger], uid=20189, componentID=2000, globalorderid=1, grouporderid=1, totalcount=1
  INFO root [2018-05-21 09:03:37,150] - Components::findComponents: find dbmgr(2)...
  INFO root [2018-05-21 09:03:37,253] - Components::findComponents: found dbmgr, addr:192.168.0.111:54035
  INFO root [2018-05-21 09:03:37,253] - Components::addComponent: reset orderLog, uid=20189!
  INFO root [2018-05-21 09:03:37,253] - Components::addComponent[dbmgr], uid=20189, componentID=4000, globalorderid=1, grouporderid=1, totalcount=1
  INFO root [2018-05-21 09:03:37,253] - Components::findComponents: found dbmgr, addr:192.168.0.111:54035
  WARN root [2018-05-21 09:03:38,253] - Forwarding logs to logger[192.168.0.111:54037/0/0/0]...
 ERROR root [2018-05-21 09:07:47,585] - Components::removeComponentByChannel: logger : 2000, Abnormal exit! errorno=10054
  INFO root [2018-05-21 09:07:47,589] - The following logs sent to logger failed:
  WARN root [2018-05-21 09:07:47,589] - ==> [2018-05-21 09:07:47 582] TCPPacketReceiver::processPendingEvents(192.168.0.111:54037): Throwing REASON_GENERAL_NETWORK - WSAECONNRESET
 ERROR root [2018-05-21 09:07:47,589] - ==> [2018-05-21 09:07:47 585] Components::removeComponentByChannel: logger : 2000, Abnormal exit! errorno=10054
  WARN root [2018-05-21 09:07:47,630] - TCPPacketReceiver::processPendingEvents(192.168.0.111:54035): Throwing REASON_GENERAL_NETWORK - WSAECONNRESET
 ERROR root [2018-05-21 09:07:47,640] - Components::removeComponentByChannel: dbmgr : 4000, Abnormal exit! errorno=10054
  INFO root [2018-05-21 09:07:47,646] - Shutdowner::onShutdownBegin: shutting down(period=0, tickPeriod=1)
S_INFO root [2018-05-21 09:07:47,646] - onLoginAppShutDown()
  INFO root [2018-05-21 09:07:47,646] - Shutdowner::onShutdown: shutting down
  INFO root [2018-05-21 09:07:47,647] - Shutdowner::onShutdownEnd: shutting down
 DEBUG root [2018-05-21 09:07:47,647] - Script::uninstall_py_dlls(): PyInit_pyexpat
 DEBUG root [2018-05-21 09:07:47,647] - Script::uninstall_py_dlls(): PyInit__socket
 DEBUG root [2018-05-21 09:07:47,647] - Script::uninstall_py_dlls(): PyInit__ssl
 DEBUG root [2018-05-21 09:07:47,647] - Script::uninstall_py_dlls(): PyInit__hashlib
 DEBUG root [2018-05-21 09:07:47,647] - Script::uninstall_py_dlls(): PyInit_select
 DEBUG root [2018-05-21 09:07:47,647] - Script::uninstall_py_dlls(): PyInit__ctypes
 DEBUG root [2018-05-21 09:07:47,647] - Script::uninstall_py_dlls(): PyInit__elementtree
 DEBUG root [2018-05-21 09:07:47,648] - Script::uninstall_py_dlls(): PyInit_unicodedata
  INFO root [2018-05-21 09:07:47,659] - Script::uninstall(): is successfully!
 DEBUG root [2018-05-21 09:07:47,659] - ThreadPool::destroy(): starting size 2.
  WARN root [2018-05-21 09:07:47,960] - ThreadPool::destroy(): waiting for thread(2)[0x489cd48,0x489d228,], try=1
  INFO root [2018-05-21 15:15:13,079] - 
  INFO root [2018-05-21 15:15:13,101] - -----------------------------------------------------------------------------------------


  INFO root [2018-05-21 15:15:13,102] - Resmgr::initialize: KBE_ROOT=D:/noah/ChessWarrior/server/
  INFO root [2018-05-21 15:15:13,103] - Resmgr::initialize: KBE_RES_PATH=D:/noah/ChessWarrior/server/kbe/res/;D:/noah/ChessWarrior/server/chess_assets/;D:/noah/ChessWarrior/server/chess_assets/scripts/;D:/noah/ChessWarrior/server/chess_assets/res/
  INFO root [2018-05-21 15:15:13,103] - Resmgr::initialize: KBE_BIN_PATH=D:/noah/ChessWarrior/server/kbe/bin/server/
  INFO root [2018-05-21 15:15:13,107] - NetworkInterface::initialize(EXTERNAL): bound to all interfaces with default route interface on 192.168.0.111 ( 192.168.0.111:20013 )
  INFO root [2018-05-21 15:15:13,107] - NetworkInterface::initialize(EXTERNAL): address 192.168.0.111:20013, SOMAXCONN=128.
  INFO root [2018-05-21 15:15:13,108] - NetworkInterface::initialize(INTERNAL): bound to all interfaces with default route interface on 192.168.0.111 ( 192.168.0.111:52547 )
  INFO root [2018-05-21 15:15:13,108] - NetworkInterface::initialize(INTERNAL): address 192.168.0.111:52547, SOMAXCONN=128.
  INFO root [2018-05-21 15:15:13,108] - server-configs:
  INFO root [2018-05-21 15:15:13,108] - 	internalAddr : 192.168.0.111:52547
  INFO root [2018-05-21 15:15:13,109] - 	externalAddr : 192.168.0.111:20013
  INFO root [2018-05-21 15:15:13,109] - 	componentID : 9000
  INFO root [2018-05-21 15:15:13,291] - Components::findLogger: found logger, addr:192.168.0.111:52413
  INFO root [2018-05-21 15:15:13,291] - Components::addComponent: reset orderLog, uid=27286!
  INFO root [2018-05-21 15:15:13,292] - Components::addComponent[logger], uid=27286, componentID=2000, globalorderid=1, grouporderid=1, totalcount=1
  INFO root [2018-05-21 15:15:13,298] - ---- loginapp Version: 2.0.0. ScriptVersion: 0.1.0. Protocol: B20F257E332779D9E1EA7D34638E6146. Config: Debug X86. Built: 15:50:13 May 15 2018. AppID: 9000. UID: 27286. PID: 13880 ----
  INFO root [2018-05-21 15:15:13,298] - Server : Intel(R) Core(TM) i5-7500 CPU @ 3.40GHz with 2097151kB RAM
  INFO root [2018-05-21 15:15:13,298] - ThreadPool::createThreadPool: creating  threadpool...
  INFO root [2018-05-21 15:15:13,298] - ThreadPool::createThreadPool: successfully(2), newThreadCount=1, normalMaxThreadCount=2, maxThreadCount=8
 DEBUG root [2018-05-21 15:15:13,434] - Script::install_py_dlls(): PyInit_pyexpat
 DEBUG root [2018-05-21 15:15:13,434] - Script::install_py_dlls(): PyInit__socket
 DEBUG root [2018-05-21 15:15:13,435] - Script::install_py_dlls(): PyInit__ssl
 DEBUG root [2018-05-21 15:15:13,438] - Script::install_py_dlls(): PyInit__hashlib
 DEBUG root [2018-05-21 15:15:13,439] - Script::install_py_dlls(): PyInit_select
 DEBUG root [2018-05-21 15:15:13,439] - Script::install_py_dlls(): PyInit__ctypes
 DEBUG root [2018-05-21 15:15:13,439] - Script::install_py_dlls(): PyInit__elementtree
 DEBUG root [2018-05-21 15:15:13,480] - Script::install_py_dlls(): PyInit_unicodedata
  INFO root [2018-05-21 15:15:13,517] - Script::install(): is successfully, Python=(3.4.2 (default, May 15 2018, 15:50:12) [MSC v.1900 32 bit (Intel)])!
  INFO root [2018-05-21 15:15:13,517] - Script::install(): KBExtra is successfully!
S_INFO root [2018-05-21 15:15:13,524] - onLoginAppReady: bootstrapGroupIndex=0, bootstrapGlobalIndex=0
  INFO root [2018-05-21 15:15:13,524] - TelnetServer server is running on port 31000
  INFO root [2018-05-21 15:15:13,994] - ---- loginapp is running ----
 DEBUG root [2018-05-21 15:15:13,995] - Components::process(): Request for the process of identity...
  WARN root [2018-05-21 15:15:14,320] - BundleBroadcast::BundleBroadcast: Cannot bind to port 20695, errorno=10048
  WARN root [2018-05-21 15:15:20,480] - Forwarding logs to logger[192.168.0.111:52413/0/0/0]...
 ERROR root [2018-05-21 15:17:36,211] - Components::removeComponentByChannel: dbmgr : 4000, Abnormal exit! errorno=10054
  INFO root [2018-05-21 15:21:32,021] - 
  INFO root [2018-05-21 15:21:32,021] - -----------------------------------------------------------------------------------------


  INFO root [2018-05-21 15:21:32,028] - Resmgr::initialize: KBE_ROOT=D:/noah/ChessWarrior/server/
  INFO root [2018-05-21 15:21:32,028] - Resmgr::initialize: KBE_RES_PATH=D:/noah/ChessWarrior/server/kbe/res/;D:/noah/ChessWarrior/server/chess_assets/;D:/noah/ChessWarrior/server/chess_assets/scripts/;D:/noah/ChessWarrior/server/chess_assets/res/
  INFO root [2018-05-21 15:21:32,029] - Resmgr::initialize: KBE_BIN_PATH=D:/noah/ChessWarrior/server/kbe/bin/server/
  INFO root [2018-05-21 15:21:32,032] - NetworkInterface::initialize(EXTERNAL): bound to all interfaces with default route interface on 192.168.0.111 ( 192.168.0.111:20013 )
  INFO root [2018-05-21 15:21:32,032] - NetworkInterface::initialize(EXTERNAL): address 192.168.0.111:20013, SOMAXCONN=128.
  INFO root [2018-05-21 15:21:32,033] - NetworkInterface::initialize(INTERNAL): bound to all interfaces with default route interface on 192.168.0.111 ( 192.168.0.111:53094 )
  INFO root [2018-05-21 15:21:32,033] - NetworkInterface::initialize(INTERNAL): address 192.168.0.111:53094, SOMAXCONN=128.
  INFO root [2018-05-21 15:21:32,033] - server-configs:
  INFO root [2018-05-21 15:21:32,034] - 	internalAddr : 192.168.0.111:53094
  INFO root [2018-05-21 15:21:32,034] - 	externalAddr : 192.168.0.111:20013
  INFO root [2018-05-21 15:21:32,034] - 	componentID : 9000
  WARN root [2018-05-21 15:21:32,533] - BundleBroadcast::BundleBroadcast: Cannot bind to port 20278, errorno=10048
  INFO root [2018-05-21 15:21:32,537] - ---- loginapp Version: 2.0.0. ScriptVersion: 0.1.0. Protocol: B20F257E332779D9E1EA7D34638E6146. Config: Debug X86. Built: 15:50:13 May 15 2018. AppID: 9000. UID: 28729. PID: 16256 ----
  INFO root [2018-05-21 15:21:32,537] - Server : Intel(R) Core(TM) i5-7500 CPU @ 3.40GHz with 2097151kB RAM
  INFO root [2018-05-21 15:21:32,537] - ThreadPool::createThreadPool: creating  threadpool...
  INFO root [2018-05-21 15:21:32,538] - ThreadPool::createThreadPool: successfully(2), newThreadCount=1, normalMaxThreadCount=2, maxThreadCount=8
 DEBUG root [2018-05-21 15:21:32,668] - Script::install_py_dlls(): PyInit_pyexpat
 DEBUG root [2018-05-21 15:21:32,669] - Script::install_py_dlls(): PyInit__socket
 DEBUG root [2018-05-21 15:21:32,669] - Script::install_py_dlls(): PyInit__ssl
 DEBUG root [2018-05-21 15:21:32,672] - Script::install_py_dlls(): PyInit__hashlib
 DEBUG root [2018-05-21 15:21:32,672] - Script::install_py_dlls(): PyInit_select
 DEBUG root [2018-05-21 15:21:32,672] - Script::install_py_dlls(): PyInit__ctypes
 DEBUG root [2018-05-21 15:21:32,673] - Script::install_py_dlls(): PyInit__elementtree
 DEBUG root [2018-05-21 15:21:32,709] - Script::install_py_dlls(): PyInit_unicodedata
  INFO root [2018-05-21 15:21:32,747] - Script::install(): is successfully, Python=(3.4.2 (default, May 15 2018, 15:50:12) [MSC v.1900 32 bit (Intel)])!
  INFO root [2018-05-21 15:21:32,747] - Script::install(): KBExtra is successfully!
S_INFO root [2018-05-21 15:21:32,749] - onLoginAppReady: bootstrapGroupIndex=0, bootstrapGlobalIndex=0
  INFO root [2018-05-21 15:21:32,750] - TelnetServer server is running on port 31000
  INFO root [2018-05-21 15:21:33,141] - ---- loginapp is running ----
 DEBUG root [2018-05-21 15:21:33,141] - Components::process(): Request for the process of identity...
  INFO root [2018-05-21 15:21:40,290] - Components::findComponents: find logger(1)...
  INFO root [2018-05-21 15:21:40,368] - Components::findComponents: found logger, addr:192.168.0.111:53101
  INFO root [2018-05-21 15:21:40,368] - Components::addComponent: reset orderLog, uid=28729!
  INFO root [2018-05-21 15:21:40,368] - Components::addComponent[logger], uid=28729, componentID=2000, globalorderid=1, grouporderid=1, totalcount=1
  INFO root [2018-05-21 15:21:40,370] - Components::findComponents: find dbmgr(2)...
  INFO root [2018-05-21 15:21:40,472] - Components::findComponents: found dbmgr, addr:192.168.0.111:53096
  INFO root [2018-05-21 15:21:40,472] - Components::addComponent: reset orderLog, uid=28729!
  INFO root [2018-05-21 15:21:40,472] - Components::addComponent[dbmgr], uid=28729, componentID=4000, globalorderid=1, grouporderid=1, totalcount=1
  INFO root [2018-05-21 15:21:40,472] - Components::findComponents: found dbmgr, addr:192.168.0.111:53096
  WARN root [2018-05-21 15:21:41,473] - Forwarding logs to logger[192.168.0.111:53101/0/0/0]...
 ERROR root [2018-05-21 15:21:46,616] - Components::removeComponentByChannel: logger : 2000, Abnormal exit! errorno=10054
  INFO root [2018-05-21 15:21:46,617] - The following logs sent to logger failed:
  WARN root [2018-05-21 15:21:46,641] - ==> [2018-05-21 15:21:46 615] TCPPacketReceiver::processPendingEvents(192.168.0.111:53101): Throwing REASON_GENERAL_NETWORK - WSAECONNRESET
 ERROR root [2018-05-21 15:21:46,641] - ==> [2018-05-21 15:21:46 617] Components::removeComponentByChannel: logger : 2000, Abnormal exit! errorno=10054
  WARN root [2018-05-21 15:21:46,641] - TCPPacketReceiver::processPendingEvents(192.168.0.111:53096): Throwing REASON_GENERAL_NETWORK - WSAECONNRESET
 ERROR root [2018-05-21 15:21:46,643] - Components::removeComponentByChannel: dbmgr : 4000, Abnormal exit! errorno=10054
  INFO root [2018-05-21 15:21:46,646] - Shutdowner::onShutdownBegin: shutting down(period=0, tickPeriod=1)
S_INFO root [2018-05-21 15:21:46,646] - onLoginAppShutDown()
  INFO root [2018-05-21 15:21:46,646] - Shutdowner::onShutdown: shutting down
  INFO root [2018-05-21 15:21:46,647] - Shutdowner::onShutdownEnd: shutting down
 DEBUG root [2018-05-21 15:21:46,647] - Script::uninstall_py_dlls(): PyInit_pyexpat
 DEBUG root [2018-05-21 15:21:46,647] - Script::uninstall_py_dlls(): PyInit__socket
 DEBUG root [2018-05-21 15:21:46,647] - Script::uninstall_py_dlls(): PyInit__ssl
 DEBUG root [2018-05-21 15:21:46,647] - Script::uninstall_py_dlls(): PyInit__hashlib
 DEBUG root [2018-05-21 15:21:46,647] - Script::uninstall_py_dlls(): PyInit_select
 DEBUG root [2018-05-21 15:21:46,647] - Script::uninstall_py_dlls(): PyInit__ctypes
 DEBUG root [2018-05-21 15:21:46,647] - Script::uninstall_py_dlls(): PyInit__elementtree
 DEBUG root [2018-05-21 15:21:46,647] - Script::uninstall_py_dlls(): PyInit_unicodedata
  INFO root [2018-05-21 15:21:46,658] - Script::uninstall(): is successfully!
 DEBUG root [2018-05-21 15:21:46,658] - ThreadPool::destroy(): starting size 2.
  INFO root [2018-05-21 15:23:44,832] - 
  INFO root [2018-05-21 15:23:44,833] - -----------------------------------------------------------------------------------------


  INFO root [2018-05-21 15:23:44,834] - Resmgr::initialize: KBE_ROOT=D:/noah/ChessWarrior/server/
  INFO root [2018-05-21 15:23:44,834] - Resmgr::initialize: KBE_RES_PATH=D:/noah/ChessWarrior/server/kbe/res/;D:/noah/ChessWarrior/server/chess_assets/;D:/noah/ChessWarrior/server/chess_assets/scripts/;D:/noah/ChessWarrior/server/chess_assets/res/
  INFO root [2018-05-21 15:23:44,834] - Resmgr::initialize: KBE_BIN_PATH=D:/noah/ChessWarrior/server/kbe/bin/server/
  INFO root [2018-05-21 15:23:44,841] - NetworkInterface::initialize(EXTERNAL): bound to all interfaces with default route interface on 192.168.0.111 ( 192.168.0.111:20013 )
  INFO root [2018-05-21 15:23:44,842] - NetworkInterface::initialize(EXTERNAL): address 192.168.0.111:20013, SOMAXCONN=128.
  INFO root [2018-05-21 15:23:44,842] - NetworkInterface::initialize(INTERNAL): bound to all interfaces with default route interface on 192.168.0.111 ( 192.168.0.111:53533 )
  INFO root [2018-05-21 15:23:44,843] - NetworkInterface::initialize(INTERNAL): address 192.168.0.111:53533, SOMAXCONN=128.
  INFO root [2018-05-21 15:23:44,843] - server-configs:
  INFO root [2018-05-21 15:23:44,843] - 	internalAddr : 192.168.0.111:53533
  INFO root [2018-05-21 15:23:44,843] - 	externalAddr : 192.168.0.111:20013
  INFO root [2018-05-21 15:23:44,843] - 	componentID : 9000
  INFO root [2018-05-21 15:23:46,509] - ---- loginapp Version: 2.0.0. ScriptVersion: 0.1.0. Protocol: B20F257E332779D9E1EA7D34638E6146. Config: Debug X86. Built: 15:50:13 May 15 2018. AppID: 9000. UID: 29163. PID: 10432 ----
  INFO root [2018-05-21 15:23:46,520] - Server : Intel(R) Core(TM) i5-7500 CPU @ 3.40GHz with 2097151kB RAM
  INFO root [2018-05-21 15:23:46,537] - ThreadPool::createThreadPool: creating  threadpool...
  INFO root [2018-05-21 15:23:46,538] - ThreadPool::createThreadPool: successfully(2), newThreadCount=1, normalMaxThreadCount=2, maxThreadCount=8
 DEBUG root [2018-05-21 15:23:46,726] - Script::install_py_dlls(): PyInit_pyexpat
 DEBUG root [2018-05-21 15:23:46,726] - Script::install_py_dlls(): PyInit__socket
 DEBUG root [2018-05-21 15:23:46,727] - Script::install_py_dlls(): PyInit__ssl
 DEBUG root [2018-05-21 15:23:46,731] - Script::install_py_dlls(): PyInit__hashlib
 DEBUG root [2018-05-21 15:23:46,731] - Script::install_py_dlls(): PyInit_select
 DEBUG root [2018-05-21 15:23:46,731] - Script::install_py_dlls(): PyInit__ctypes
 DEBUG root [2018-05-21 15:23:46,731] - Script::install_py_dlls(): PyInit__elementtree
 DEBUG root [2018-05-21 15:23:46,795] - Script::install_py_dlls(): PyInit_unicodedata
  INFO root [2018-05-21 15:23:46,862] - Script::install(): is successfully, Python=(3.4.2 (default, May 15 2018, 15:50:12) [MSC v.1900 32 bit (Intel)])!
  INFO root [2018-05-21 15:23:46,862] - Script::install(): KBExtra is successfully!
S_INFO root [2018-05-21 15:23:46,864] - onLoginAppReady: bootstrapGroupIndex=0, bootstrapGlobalIndex=0
  INFO root [2018-05-21 15:23:46,865] - TelnetServer server is running on port 31000
  INFO root [2018-05-21 15:23:47,288] - ---- loginapp is running ----
 DEBUG root [2018-05-21 15:23:47,289] - Components::process(): Request for the process of identity...
  WARN root [2018-05-21 15:23:47,615] - BundleBroadcast::BundleBroadcast: Cannot bind to port 20687, errorno=10048
  INFO root [2018-05-21 15:23:54,785] - Components::findComponents: find logger(1)...
  INFO root [2018-05-21 15:23:54,880] - Components::findComponents: found logger, addr:192.168.0.111:53536
  INFO root [2018-05-21 15:23:54,881] - Components::addComponent: reset orderLog, uid=29163!
  INFO root [2018-05-21 15:23:54,881] - Components::addComponent[logger], uid=29163, componentID=2000, globalorderid=1, grouporderid=1, totalcount=1
  INFO root [2018-05-21 15:23:54,882] - Components::findComponents: find dbmgr(2)...
  INFO root [2018-05-21 15:23:54,984] - Components::findComponents: found dbmgr, addr:192.168.0.111:53531
  INFO root [2018-05-21 15:23:54,984] - Components::addComponent: reset orderLog, uid=29163!
  INFO root [2018-05-21 15:23:54,984] - Components::addComponent[dbmgr], uid=29163, componentID=4000, globalorderid=1, grouporderid=1, totalcount=1
  INFO root [2018-05-21 15:23:54,984] - Components::findComponents: found dbmgr, addr:192.168.0.111:53531
  WARN root [2018-05-21 15:23:55,985] - Forwarding logs to logger[192.168.0.111:53536/0/0/0]...
 ERROR root [2018-05-21 15:24:48,481] - Components::removeComponentByChannel: dbmgr : 4000, Abnormal exit! errorno=10054
  INFO root [2018-05-21 15:25:35,515] - 
  INFO root [2018-05-21 15:25:35,523] - -----------------------------------------------------------------------------------------


  INFO root [2018-05-21 15:25:35,524] - Resmgr::initialize: KBE_ROOT=D:/noah/ChessWarrior/server/
  INFO root [2018-05-21 15:25:35,526] - Resmgr::initialize: KBE_RES_PATH=D:/noah/ChessWarrior/server/kbe/res/;D:/noah/ChessWarrior/server/chess_assets/;D:/noah/ChessWarrior/server/chess_assets/scripts/;D:/noah/ChessWarrior/server/chess_assets/res/
  INFO root [2018-05-21 15:25:35,526] - Resmgr::initialize: KBE_BIN_PATH=D:/noah/ChessWarrior/server/kbe/bin/server/
  INFO root [2018-05-21 15:25:35,545] - NetworkInterface::initialize(EXTERNAL): bound to all interfaces with default route interface on 192.168.0.111 ( 192.168.0.111:20013 )
  INFO root [2018-05-21 15:25:35,545] - NetworkInterface::initialize(EXTERNAL): address 192.168.0.111:20013, SOMAXCONN=128.
  INFO root [2018-05-21 15:25:35,546] - NetworkInterface::initialize(INTERNAL): bound to all interfaces with default route interface on 192.168.0.111 ( 192.168.0.111:53745 )
  INFO root [2018-05-21 15:25:35,546] - NetworkInterface::initialize(INTERNAL): address 192.168.0.111:53745, SOMAXCONN=128.
  INFO root [2018-05-21 15:25:35,546] - server-configs:
  INFO root [2018-05-21 15:25:35,546] - 	internalAddr : 192.168.0.111:53745
  INFO root [2018-05-21 15:25:35,546] - 	externalAddr : 192.168.0.111:20013
  INFO root [2018-05-21 15:25:35,546] - 	componentID : 9000
  INFO root [2018-05-21 15:25:37,217] - ---- loginapp Version: 2.0.0. ScriptVersion: 0.1.0. Protocol: B20F257E332779D9E1EA7D34638E6146. Config: Debug X86. Built: 15:50:13 May 15 2018. AppID: 9000. UID: 29523. PID: 2496 ----
  INFO root [2018-05-21 15:25:37,227] - Server : Intel(R) Core(TM) i5-7500 CPU @ 3.40GHz with 2097151kB RAM
  INFO root [2018-05-21 15:25:37,227] - ThreadPool::createThreadPool: creating  threadpool...
  INFO root [2018-05-21 15:25:37,227] - ThreadPool::createThreadPool: successfully(2), newThreadCount=1, normalMaxThreadCount=2, maxThreadCount=8
 DEBUG root [2018-05-21 15:25:37,468] - Script::install_py_dlls(): PyInit_pyexpat
 DEBUG root [2018-05-21 15:25:37,469] - Script::install_py_dlls(): PyInit__socket
 DEBUG root [2018-05-21 15:25:37,469] - Script::install_py_dlls(): PyInit__ssl
 DEBUG root [2018-05-21 15:25:37,473] - Script::install_py_dlls(): PyInit__hashlib
 DEBUG root [2018-05-21 15:25:37,473] - Script::install_py_dlls(): PyInit_select
 DEBUG root [2018-05-21 15:25:37,473] - Script::install_py_dlls(): PyInit__ctypes
 DEBUG root [2018-05-21 15:25:37,474] - Script::install_py_dlls(): PyInit__elementtree
 DEBUG root [2018-05-21 15:25:37,520] - Script::install_py_dlls(): PyInit_unicodedata
  INFO root [2018-05-21 15:25:37,574] - Script::install(): is successfully, Python=(3.4.2 (default, May 15 2018, 15:50:12) [MSC v.1900 32 bit (Intel)])!
  INFO root [2018-05-21 15:25:37,574] - Script::install(): KBExtra is successfully!
S_INFO root [2018-05-21 15:25:37,594] - onLoginAppReady: bootstrapGroupIndex=0, bootstrapGlobalIndex=0
  INFO root [2018-05-21 15:25:37,594] - TelnetServer server is running on port 31000
  INFO root [2018-05-21 15:25:38,129] - ---- loginapp is running ----
 DEBUG root [2018-05-21 15:25:38,135] - Components::process(): Request for the process of identity...
  INFO root [2018-05-21 15:25:45,334] - Components::findComponents: find logger(1)...
  INFO root [2018-05-21 15:25:45,364] - Components::findComponents: found logger, addr:192.168.0.111:53740
  INFO root [2018-05-21 15:25:45,365] - Components::addComponent: reset orderLog, uid=29523!
  INFO root [2018-05-21 15:25:45,365] - Components::addComponent[logger], uid=29523, componentID=2000, globalorderid=1, grouporderid=1, totalcount=1
  INFO root [2018-05-21 15:25:45,366] - Components::findComponents: find dbmgr(2)...
  INFO root [2018-05-21 15:25:45,469] - Components::findComponents: found dbmgr, addr:192.168.0.111:53743
  INFO root [2018-05-21 15:25:45,470] - Components::addComponent: reset orderLog, uid=29523!
  INFO root [2018-05-21 15:25:45,470] - Components::addComponent[dbmgr], uid=29523, componentID=4000, globalorderid=1, grouporderid=1, totalcount=1
  INFO root [2018-05-21 15:25:45,470] - Components::findComponents: found dbmgr, addr:192.168.0.111:53743
  WARN root [2018-05-21 15:25:46,470] - Forwarding logs to logger[192.168.0.111:53740/0/0/0]...
 ERROR root [2018-05-21 15:26:40,422] - Components::removeComponentByChannel: logger : 2000, Abnormal exit! errorno=10054
  INFO root [2018-05-21 15:26:40,424] - The following logs sent to logger failed:
  WARN root [2018-05-21 15:26:40,424] - ==> [2018-05-21 15:26:40 417] TCPPacketReceiver::processPendingEvents(192.168.0.111:53740): Throwing REASON_GENERAL_NETWORK - WSAECONNRESET
 ERROR root [2018-05-21 15:26:40,425] - ==> [2018-05-21 15:26:40 423] Components::removeComponentByChannel: logger : 2000, Abnormal exit! errorno=10054
  WARN root [2018-05-21 15:26:40,502] - TCPPacketReceiver::processPendingEvents(192.168.0.111:53743): Throwing REASON_GENERAL_NETWORK - WSAECONNRESET
 ERROR root [2018-05-21 15:26:40,505] - Components::removeComponentByChannel: dbmgr : 4000, Abnormal exit! errorno=10054
  INFO root [2018-05-21 15:26:40,506] - Shutdowner::onShutdownBegin: shutting down(period=0, tickPeriod=1)
S_INFO root [2018-05-21 15:26:40,506] - onLoginAppShutDown()
  INFO root [2018-05-21 15:26:40,506] - Shutdowner::onShutdown: shutting down
  INFO root [2018-05-21 15:26:40,506] - Shutdowner::onShutdownEnd: shutting down
 DEBUG root [2018-05-21 15:26:40,507] - Script::uninstall_py_dlls(): PyInit_pyexpat
 DEBUG root [2018-05-21 15:26:40,507] - Script::uninstall_py_dlls(): PyInit__socket
 DEBUG root [2018-05-21 15:26:40,507] - Script::uninstall_py_dlls(): PyInit__ssl
 DEBUG root [2018-05-21 15:26:40,507] - Script::uninstall_py_dlls(): PyInit__hashlib
 DEBUG root [2018-05-21 15:26:40,507] - Script::uninstall_py_dlls(): PyInit_select
 DEBUG root [2018-05-21 15:26:40,507] - Script::uninstall_py_dlls(): PyInit__ctypes
 DEBUG root [2018-05-21 15:26:40,507] - Script::uninstall_py_dlls(): PyInit__elementtree
 DEBUG root [2018-05-21 15:26:40,507] - Script::uninstall_py_dlls(): PyInit_unicodedata
  INFO root [2018-05-21 15:26:40,518] - Script::uninstall(): is successfully!
 DEBUG root [2018-05-21 15:26:40,518] - ThreadPool::destroy(): starting size 2.
  WARN root [2018-05-21 15:26:40,819] - ThreadPool::destroy(): waiting for thread(2)[0xc25170,0xc25720,], try=1
  INFO root [2018-05-21 15:28:02,813] - 
  INFO root [2018-05-21 15:28:02,813] - -----------------------------------------------------------------------------------------


  INFO root [2018-05-21 15:28:02,814] - Resmgr::initialize: KBE_ROOT=D:/noah/ChessWarrior/server/
  INFO root [2018-05-21 15:28:02,814] - Resmgr::initialize: KBE_RES_PATH=D:/noah/ChessWarrior/server/kbe/res/;D:/noah/ChessWarrior/server/chess_assets/;D:/noah/ChessWarrior/server/chess_assets/scripts/;D:/noah/ChessWarrior/server/chess_assets/res/
  INFO root [2018-05-21 15:28:02,814] - Resmgr::initialize: KBE_BIN_PATH=D:/noah/ChessWarrior/server/kbe/bin/server/
  INFO root [2018-05-21 15:28:02,816] - NetworkInterface::initialize(EXTERNAL): bound to all interfaces with default route interface on 192.168.0.111 ( 192.168.0.111:20013 )
  INFO root [2018-05-21 15:28:02,816] - NetworkInterface::initialize(EXTERNAL): address 192.168.0.111:20013, SOMAXCONN=128.
  INFO root [2018-05-21 15:28:02,817] - NetworkInterface::initialize(INTERNAL): bound to all interfaces with default route interface on 192.168.0.111 ( 192.168.0.111:54027 )
  INFO root [2018-05-21 15:28:02,817] - NetworkInterface::initialize(INTERNAL): address 192.168.0.111:54027, SOMAXCONN=128.
  INFO root [2018-05-21 15:28:02,817] - server-configs:
  INFO root [2018-05-21 15:28:02,817] - 	internalAddr : 192.168.0.111:54027
  INFO root [2018-05-21 15:28:02,817] - 	externalAddr : 192.168.0.111:20013
  INFO root [2018-05-21 15:28:02,817] - 	componentID : 9000
  INFO root [2018-05-21 15:28:04,509] - ---- loginapp Version: 2.0.0. ScriptVersion: 0.1.0. Protocol: B20F257E332779D9E1EA7D34638E6146. Config: Debug X86. Built: 15:50:13 May 15 2018. AppID: 9000. UID: 30006. PID: 7700 ----
  INFO root [2018-05-21 15:28:04,514] - Server : Intel(R) Core(TM) i5-7500 CPU @ 3.40GHz with 2097151kB RAM
  INFO root [2018-05-21 15:28:04,521] - ThreadPool::createThreadPool: creating  threadpool...
  INFO root [2018-05-21 15:28:04,522] - ThreadPool::createThreadPool: successfully(2), newThreadCount=1, normalMaxThreadCount=2, maxThreadCount=8
 DEBUG root [2018-05-21 15:28:04,819] - Script::install_py_dlls(): PyInit_pyexpat
 DEBUG root [2018-05-21 15:28:04,819] - Script::install_py_dlls(): PyInit__socket
 DEBUG root [2018-05-21 15:28:04,820] - Script::install_py_dlls(): PyInit__ssl
 DEBUG root [2018-05-21 15:28:04,824] - Script::install_py_dlls(): PyInit__hashlib
 DEBUG root [2018-05-21 15:28:04,824] - Script::install_py_dlls(): PyInit_select
 DEBUG root [2018-05-21 15:28:04,824] - Script::install_py_dlls(): PyInit__ctypes
 DEBUG root [2018-05-21 15:28:04,825] - Script::install_py_dlls(): PyInit__elementtree
 DEBUG root [2018-05-21 15:28:04,896] - Script::install_py_dlls(): PyInit_unicodedata
  INFO root [2018-05-21 15:28:04,944] - Script::install(): is successfully, Python=(3.4.2 (default, May 15 2018, 15:50:12) [MSC v.1900 32 bit (Intel)])!
  INFO root [2018-05-21 15:28:04,945] - Script::install(): KBExtra is successfully!
S_INFO root [2018-05-21 15:28:04,947] - onLoginAppReady: bootstrapGroupIndex=0, bootstrapGlobalIndex=0
  INFO root [2018-05-21 15:28:04,948] - TelnetServer server is running on port 31000
  INFO root [2018-05-21 15:28:05,564] - ---- loginapp is running ----
 DEBUG root [2018-05-21 15:28:05,575] - Components::process(): Request for the process of identity...
  WARN root [2018-05-21 15:28:09,058] - BundleBroadcast::BundleBroadcast: Cannot bind to port 20514, errorno=10048
  INFO root [2018-05-21 15:28:13,107] - Components::findComponents: find logger(1)...
  INFO root [2018-05-21 15:28:13,184] - Components::findComponents: found logger, addr:192.168.0.111:54035
  INFO root [2018-05-21 15:28:13,184] - Components::addComponent: reset orderLog, uid=30006!
  INFO root [2018-05-21 15:28:13,184] - Components::addComponent[logger], uid=30006, componentID=2000, globalorderid=1, grouporderid=1, totalcount=1
  INFO root [2018-05-21 15:28:13,186] - Components::findComponents: find dbmgr(2)...
  INFO root [2018-05-21 15:28:13,291] - Components::findComponents: found dbmgr, addr:192.168.0.111:54033
  INFO root [2018-05-21 15:28:13,291] - Components::addComponent: reset orderLog, uid=30006!
  INFO root [2018-05-21 15:28:13,291] - Components::addComponent[dbmgr], uid=30006, componentID=4000, globalorderid=1, grouporderid=1, totalcount=1
  INFO root [2018-05-21 15:28:13,293] - Components::findComponents: found dbmgr, addr:192.168.0.111:54033
  WARN root [2018-05-21 15:28:14,293] - Forwarding logs to logger[192.168.0.111:54035/0/0/0]...
 ERROR root [2018-05-21 15:28:54,255] - Components::removeComponentByChannel: logger : 2000, Abnormal exit! errorno=10054
  INFO root [2018-05-21 15:28:54,258] - The following logs sent to logger failed:
  WARN root [2018-05-21 15:28:54,258] - ==> [2018-05-21 15:28:54 255] TCPPacketReceiver::processPendingEvents(192.168.0.111:54035): Throwing REASON_GENERAL_NETWORK - WSAECONNRESET
 ERROR root [2018-05-21 15:28:54,258] - ==> [2018-05-21 15:28:54 256] Components::removeComponentByChannel: logger : 2000, Abnormal exit! errorno=10054
  WARN root [2018-05-21 15:28:54,281] - TCPPacketReceiver::processPendingEvents(192.168.0.111:54033): Throwing REASON_GENERAL_NETWORK - WSAECONNRESET
 ERROR root [2018-05-21 15:28:54,287] - Components::removeComponentByChannel: dbmgr : 4000, Abnormal exit! errorno=10054
  INFO root [2018-05-21 15:28:54,290] - Shutdowner::onShutdownBegin: shutting down(period=0, tickPeriod=1)
S_INFO root [2018-05-21 15:28:54,290] - onLoginAppShutDown()
  INFO root [2018-05-21 15:28:54,290] - Shutdowner::onShutdown: shutting down
  INFO root [2018-05-21 15:28:54,290] - Shutdowner::onShutdownEnd: shutting down
 DEBUG root [2018-05-21 15:28:54,291] - Script::uninstall_py_dlls(): PyInit_pyexpat
 DEBUG root [2018-05-21 15:28:54,291] - Script::uninstall_py_dlls(): PyInit__socket
 DEBUG root [2018-05-21 15:28:54,291] - Script::uninstall_py_dlls(): PyInit__ssl
 DEBUG root [2018-05-21 15:28:54,291] - Script::uninstall_py_dlls(): PyInit__hashlib
 DEBUG root [2018-05-21 15:28:54,291] - Script::uninstall_py_dlls(): PyInit_select
 DEBUG root [2018-05-21 15:28:54,291] - Script::uninstall_py_dlls(): PyInit__ctypes
 DEBUG root [2018-05-21 15:28:54,292] - Script::uninstall_py_dlls(): PyInit__elementtree
 DEBUG root [2018-05-21 15:28:54,292] - Script::uninstall_py_dlls(): PyInit_unicodedata
  INFO root [2018-05-21 15:28:54,301] - Script::uninstall(): is successfully!
 DEBUG root [2018-05-21 15:28:54,302] - ThreadPool::destroy(): starting size 2.
  WARN root [2018-05-21 15:28:54,602] - ThreadPool::destroy(): waiting for thread(2)[0x9ac958,0x9acaf8,], try=1
  INFO root [2018-05-21 15:47:32,562] - 
  INFO root [2018-05-21 15:47:32,563] - -----------------------------------------------------------------------------------------


  INFO root [2018-05-21 15:47:32,563] - Resmgr::initialize: KBE_ROOT=D:/noah/ChessWarrior/server/
  INFO root [2018-05-21 15:47:32,564] - Resmgr::initialize: KBE_RES_PATH=D:/noah/ChessWarrior/server/kbe/res/;D:/noah/ChessWarrior/server/chess_assets/;D:/noah/ChessWarrior/server/chess_assets/scripts/;D:/noah/ChessWarrior/server/chess_assets/res/
  INFO root [2018-05-21 15:47:32,564] - Resmgr::initialize: KBE_BIN_PATH=D:/noah/ChessWarrior/server/kbe/bin/server/
  INFO root [2018-05-21 15:47:32,565] - NetworkInterface::initialize(EXTERNAL): bound to all interfaces with default route interface on 192.168.0.111 ( 192.168.0.111:20013 )
  INFO root [2018-05-21 15:47:32,565] - NetworkInterface::initialize(EXTERNAL): address 192.168.0.111:20013, SOMAXCONN=128.
  INFO root [2018-05-21 15:47:32,566] - NetworkInterface::initialize(INTERNAL): bound to all interfaces with default route interface on 192.168.0.111 ( 192.168.0.111:56052 )
  INFO root [2018-05-21 15:47:32,566] - NetworkInterface::initialize(INTERNAL): address 192.168.0.111:56052, SOMAXCONN=128.
  INFO root [2018-05-21 15:47:32,567] - server-configs:
  INFO root [2018-05-21 15:47:32,567] - 	internalAddr : 192.168.0.111:56052
  INFO root [2018-05-21 15:47:32,567] - 	externalAddr : 192.168.0.111:20013
  INFO root [2018-05-21 15:47:32,567] - 	componentID : 9000
  INFO root [2018-05-21 15:47:34,222] - ---- loginapp Version: 2.0.0. ScriptVersion: 0.1.0. Protocol: B20F257E332779D9E1EA7D34638E6146. Config: Debug X86. Built: 15:50:13 May 15 2018. AppID: 9000. UID: 1055. PID: 16696 ----
  INFO root [2018-05-21 15:47:34,223] - Server : Intel(R) Core(TM) i5-7500 CPU @ 3.40GHz with 2097151kB RAM
  INFO root [2018-05-21 15:47:34,223] - ThreadPool::createThreadPool: creating  threadpool...
  INFO root [2018-05-21 15:47:34,223] - ThreadPool::createThreadPool: successfully(2), newThreadCount=1, normalMaxThreadCount=2, maxThreadCount=8
 DEBUG root [2018-05-21 15:47:34,360] - Script::install_py_dlls(): PyInit_pyexpat
 DEBUG root [2018-05-21 15:47:34,361] - Script::install_py_dlls(): PyInit__socket
 DEBUG root [2018-05-21 15:47:34,361] - Script::install_py_dlls(): PyInit__ssl
 DEBUG root [2018-05-21 15:47:34,364] - Script::install_py_dlls(): PyInit__hashlib
 DEBUG root [2018-05-21 15:47:34,364] - Script::install_py_dlls(): PyInit_select
 DEBUG root [2018-05-21 15:47:34,364] - Script::install_py_dlls(): PyInit__ctypes
 DEBUG root [2018-05-21 15:47:34,365] - Script::install_py_dlls(): PyInit__elementtree
 DEBUG root [2018-05-21 15:47:34,408] - Script::install_py_dlls(): PyInit_unicodedata
  INFO root [2018-05-21 15:47:34,451] - Script::install(): is successfully, Python=(3.4.2 (default, May 15 2018, 15:50:12) [MSC v.1900 32 bit (Intel)])!
  INFO root [2018-05-21 15:47:34,451] - Script::install(): KBExtra is successfully!
S_INFO root [2018-05-21 15:47:34,453] - onLoginAppReady: bootstrapGroupIndex=0, bootstrapGlobalIndex=0
  INFO root [2018-05-21 15:47:34,453] - TelnetServer server is running on port 31000
  INFO root [2018-05-21 15:47:34,875] - ---- loginapp is running ----
 DEBUG root [2018-05-21 15:47:34,875] - Components::process(): Request for the process of identity...
  INFO root [2018-05-21 15:48:00,755] - 
  INFO root [2018-05-21 15:48:00,757] - -----------------------------------------------------------------------------------------


  INFO root [2018-05-21 15:48:00,760] - Resmgr::initialize: KBE_ROOT=D:/noah/ChessWarrior/server/
  INFO root [2018-05-21 15:48:00,760] - Resmgr::initialize: KBE_RES_PATH=D:/noah/ChessWarrior/server/kbe/res/;D:/noah/ChessWarrior/server/chess_assets/;D:/noah/ChessWarrior/server/chess_assets/scripts/;D:/noah/ChessWarrior/server/chess_assets/res/
  INFO root [2018-05-21 15:48:00,760] - Resmgr::initialize: KBE_BIN_PATH=D:/noah/ChessWarrior/server/kbe/bin/server/
  INFO root [2018-05-21 15:48:00,762] - NetworkInterface::initialize(EXTERNAL): bound to all interfaces with default route interface on 192.168.0.111 ( 192.168.0.111:20013 )
  INFO root [2018-05-21 15:48:00,762] - NetworkInterface::initialize(EXTERNAL): address 192.168.0.111:20013, SOMAXCONN=128.
  INFO root [2018-05-21 15:48:00,763] - NetworkInterface::initialize(INTERNAL): bound to all interfaces with default route interface on 192.168.0.111 ( 192.168.0.111:56110 )
  INFO root [2018-05-21 15:48:00,763] - NetworkInterface::initialize(INTERNAL): address 192.168.0.111:56110, SOMAXCONN=128.
  INFO root [2018-05-21 15:48:00,763] - server-configs:
  INFO root [2018-05-21 15:48:00,764] - 	internalAddr : 192.168.0.111:56110
  INFO root [2018-05-21 15:48:00,764] - 	externalAddr : 192.168.0.111:20013
  INFO root [2018-05-21 15:48:00,764] - 	componentID : 9000
  INFO root [2018-05-21 15:48:02,422] - ---- loginapp Version: 2.0.0. ScriptVersion: 0.1.0. Protocol: B20F257E332779D9E1EA7D34638E6146. Config: Debug X86. Built: 15:50:13 May 15 2018. AppID: 9000. UID: 1150. PID: 11420 ----
  INFO root [2018-05-21 15:48:02,422] - Server : Intel(R) Core(TM) i5-7500 CPU @ 3.40GHz with 2097151kB RAM
  INFO root [2018-05-21 15:48:02,422] - ThreadPool::createThreadPool: creating  threadpool...
  INFO root [2018-05-21 15:48:02,423] - ThreadPool::createThreadPool: successfully(2), newThreadCount=1, normalMaxThreadCount=2, maxThreadCount=8
 DEBUG root [2018-05-21 15:48:02,558] - Script::install_py_dlls(): PyInit_pyexpat
 DEBUG root [2018-05-21 15:48:02,558] - Script::install_py_dlls(): PyInit__socket
 DEBUG root [2018-05-21 15:48:02,559] - Script::install_py_dlls(): PyInit__ssl
 DEBUG root [2018-05-21 15:48:02,562] - Script::install_py_dlls(): PyInit__hashlib
 DEBUG root [2018-05-21 15:48:02,563] - Script::install_py_dlls(): PyInit_select
 DEBUG root [2018-05-21 15:48:02,563] - Script::install_py_dlls(): PyInit__ctypes
 DEBUG root [2018-05-21 15:48:02,564] - Script::install_py_dlls(): PyInit__elementtree
 DEBUG root [2018-05-21 15:48:02,613] - Script::install_py_dlls(): PyInit_unicodedata
  INFO root [2018-05-21 15:48:02,663] - Script::install(): is successfully, Python=(3.4.2 (default, May 15 2018, 15:50:12) [MSC v.1900 32 bit (Intel)])!
  INFO root [2018-05-21 15:48:02,663] - Script::install(): KBExtra is successfully!
S_INFO root [2018-05-21 15:48:02,666] - onLoginAppReady: bootstrapGroupIndex=0, bootstrapGlobalIndex=0
  INFO root [2018-05-21 15:48:02,667] - TelnetServer server is running on port 31000
  INFO root [2018-05-21 15:48:03,059] - ---- loginapp is running ----
 DEBUG root [2018-05-21 15:48:03,060] - Components::process(): Request for the process of identity...
  INFO root [2018-05-21 15:48:10,261] - Components::findComponents: find logger(1)...
  INFO root [2018-05-21 15:48:10,362] - Components::findComponents: found logger, addr:192.168.0.111:56111
  INFO root [2018-05-21 15:48:10,362] - Components::addComponent: reset orderLog, uid=1150!
  INFO root [2018-05-21 15:48:10,362] - Components::addComponent[logger], uid=1150, componentID=2000, globalorderid=1, grouporderid=1, totalcount=1
  INFO root [2018-05-21 15:48:10,363] - Components::findComponents: find dbmgr(2)...
  INFO root [2018-05-21 15:48:10,466] - Components::findComponents: found dbmgr, addr:192.168.0.111:56112
  INFO root [2018-05-21 15:48:10,466] - Components::addComponent: reset orderLog, uid=1150!
  INFO root [2018-05-21 15:48:10,466] - Components::addComponent[dbmgr], uid=1150, componentID=4000, globalorderid=1, grouporderid=1, totalcount=1
  INFO root [2018-05-21 15:48:10,466] - Components::findComponents: found dbmgr, addr:192.168.0.111:56112
  WARN root [2018-05-21 15:48:11,467] - Forwarding logs to logger[192.168.0.111:56111/0/0/0]...
 ERROR root [2018-05-21 15:48:15,614] - Components::removeComponentByChannel: logger : 2000, Abnormal exit! errorno=10054
  INFO root [2018-05-21 15:48:15,616] - The following logs sent to logger failed:
  WARN root [2018-05-21 15:48:15,616] - ==> [2018-05-21 15:48:15 611] TCPPacketReceiver::processPendingEvents(192.168.0.111:56111): Throwing REASON_GENERAL_NETWORK - WSAECONNRESET
 ERROR root [2018-05-21 15:48:15,616] - ==> [2018-05-21 15:48:15 614] Components::removeComponentByChannel: logger : 2000, Abnormal exit! errorno=10054
  WARN root [2018-05-21 15:48:15,640] - TCPPacketReceiver::processPendingEvents(192.168.0.111:56112): Throwing REASON_GENERAL_NETWORK - WSAECONNRESET
 ERROR root [2018-05-21 15:48:15,656] - Components::removeComponentByChannel: dbmgr : 4000, Abnormal exit! errorno=10054
  INFO root [2018-05-21 15:48:15,659] - Shutdowner::onShutdownBegin: shutting down(period=0, tickPeriod=1)
S_INFO root [2018-05-21 15:48:15,659] - onLoginAppShutDown()
  INFO root [2018-05-21 15:48:15,659] - Shutdowner::onShutdown: shutting down
  INFO root [2018-05-21 15:48:15,660] - Shutdowner::onShutdownEnd: shutting down
 DEBUG root [2018-05-21 15:48:15,660] - Script::uninstall_py_dlls(): PyInit_pyexpat
 DEBUG root [2018-05-21 15:48:15,660] - Script::uninstall_py_dlls(): PyInit__socket
 DEBUG root [2018-05-21 15:48:15,660] - Script::uninstall_py_dlls(): PyInit__ssl
 DEBUG root [2018-05-21 15:48:15,660] - Script::uninstall_py_dlls(): PyInit__hashlib
 DEBUG root [2018-05-21 15:48:15,660] - Script::uninstall_py_dlls(): PyInit_select
 DEBUG root [2018-05-21 15:48:15,660] - Script::uninstall_py_dlls(): PyInit__ctypes
 DEBUG root [2018-05-21 15:48:15,660] - Script::uninstall_py_dlls(): PyInit__elementtree
 DEBUG root [2018-05-21 15:48:15,661] - Script::uninstall_py_dlls(): PyInit_unicodedata
  INFO root [2018-05-21 15:48:15,671] - Script::uninstall(): is successfully!
 DEBUG root [2018-05-21 15:48:15,671] - ThreadPool::destroy(): starting size 2.
  WARN root [2018-05-21 15:48:15,972] - ThreadPool::destroy(): waiting for thread(2)[0x8b9240,0x8b9038,], try=1
  INFO root [2018-05-21 15:49:07,892] - 
  INFO root [2018-05-21 15:49:07,892] - -----------------------------------------------------------------------------------------


  INFO root [2018-05-21 15:49:07,893] - Resmgr::initialize: KBE_ROOT=D:/noah/ChessWarrior/server/
  INFO root [2018-05-21 15:49:07,893] - Resmgr::initialize: KBE_RES_PATH=D:/noah/ChessWarrior/server/kbe/res/;D:/noah/ChessWarrior/server/chess_assets/;D:/noah/ChessWarrior/server/chess_assets/scripts/;D:/noah/ChessWarrior/server/chess_assets/res/
  INFO root [2018-05-21 15:49:07,893] - Resmgr::initialize: KBE_BIN_PATH=D:/noah/ChessWarrior/server/kbe/bin/server/
  INFO root [2018-05-21 15:49:07,895] - NetworkInterface::initialize(EXTERNAL): bound to all interfaces with default route interface on 192.168.0.111 ( 192.168.0.111:20013 )
  INFO root [2018-05-21 15:49:07,895] - NetworkInterface::initialize(EXTERNAL): address 192.168.0.111:20013, SOMAXCONN=128.
  INFO root [2018-05-21 15:49:07,896] - NetworkInterface::initialize(INTERNAL): bound to all interfaces with default route interface on 192.168.0.111 ( 192.168.0.111:56228 )
  INFO root [2018-05-21 15:49:07,896] - NetworkInterface::initialize(INTERNAL): address 192.168.0.111:56228, SOMAXCONN=128.
  INFO root [2018-05-21 15:49:07,896] - server-configs:
  INFO root [2018-05-21 15:49:07,896] - 	internalAddr : 192.168.0.111:56228
  INFO root [2018-05-21 15:49:07,897] - 	externalAddr : 192.168.0.111:20013
  INFO root [2018-05-21 15:49:07,897] - 	componentID : 9000
  INFO root [2018-05-21 15:49:09,543] - ---- loginapp Version: 2.0.0. ScriptVersion: 0.1.0. Protocol: B20F257E332779D9E1EA7D34638E6146. Config: Debug X86. Built: 15:50:13 May 15 2018. AppID: 9000. UID: 1369. PID: 15204 ----
  INFO root [2018-05-21 15:49:09,545] - Server : Intel(R) Core(TM) i5-7500 CPU @ 3.40GHz with 2097151kB RAM
  INFO root [2018-05-21 15:49:09,545] - ThreadPool::createThreadPool: creating  threadpool...
  INFO root [2018-05-21 15:49:09,545] - ThreadPool::createThreadPool: successfully(2), newThreadCount=1, normalMaxThreadCount=2, maxThreadCount=8
 DEBUG root [2018-05-21 15:49:09,679] - Script::install_py_dlls(): PyInit_pyexpat
 DEBUG root [2018-05-21 15:49:09,679] - Script::install_py_dlls(): PyInit__socket
 DEBUG root [2018-05-21 15:49:09,680] - Script::install_py_dlls(): PyInit__ssl
 DEBUG root [2018-05-21 15:49:09,684] - Script::install_py_dlls(): PyInit__hashlib
 DEBUG root [2018-05-21 15:49:09,684] - Script::install_py_dlls(): PyInit_select
 DEBUG root [2018-05-21 15:49:09,684] - Script::install_py_dlls(): PyInit__ctypes
 DEBUG root [2018-05-21 15:49:09,685] - Script::install_py_dlls(): PyInit__elementtree
 DEBUG root [2018-05-21 15:49:09,725] - Script::install_py_dlls(): PyInit_unicodedata
  INFO root [2018-05-21 15:49:09,769] - Script::install(): is successfully, Python=(3.4.2 (default, May 15 2018, 15:50:12) [MSC v.1900 32 bit (Intel)])!
  INFO root [2018-05-21 15:49:09,769] - Script::install(): KBExtra is successfully!
S_INFO root [2018-05-21 15:49:09,771] - onLoginAppReady: bootstrapGroupIndex=0, bootstrapGlobalIndex=0
  INFO root [2018-05-21 15:49:09,771] - TelnetServer server is running on port 31000
  INFO root [2018-05-21 15:49:10,236] - ---- loginapp is running ----
 DEBUG root [2018-05-21 15:49:10,252] - Components::process(): Request for the process of identity...
  WARN root [2018-05-21 15:49:13,756] - BundleBroadcast::BundleBroadcast: Cannot bind to port 20734, errorno=10048
  INFO root [2018-05-21 15:49:17,811] - Components::findComponents: find logger(1)...
  INFO root [2018-05-21 15:49:17,916] - Components::findComponents: found logger, addr:192.168.0.111:56233
  INFO root [2018-05-21 15:49:17,916] - Components::addComponent: reset orderLog, uid=1369!
  INFO root [2018-05-21 15:49:17,917] - Components::addComponent[logger], uid=1369, componentID=2000, globalorderid=1, grouporderid=1, totalcount=1
  INFO root [2018-05-21 15:49:17,918] - Components::findComponents: find dbmgr(2)...
  INFO root [2018-05-21 15:49:18,022] - Components::findComponents: found dbmgr, addr:192.168.0.111:56234
  INFO root [2018-05-21 15:49:18,022] - Components::addComponent: reset orderLog, uid=1369!
  INFO root [2018-05-21 15:49:18,022] - Components::addComponent[dbmgr], uid=1369, componentID=4000, globalorderid=1, grouporderid=1, totalcount=1
  INFO root [2018-05-21 15:49:18,022] - Components::findComponents: found dbmgr, addr:192.168.0.111:56234
  INFO root [2018-05-21 15:50:21,942] - 
  INFO root [2018-05-21 15:50:21,945] - -----------------------------------------------------------------------------------------


  INFO root [2018-05-21 15:50:21,946] - Resmgr::initialize: KBE_ROOT=D:/noah/ChessWarrior/server/
  INFO root [2018-05-21 15:50:21,946] - Resmgr::initialize: KBE_RES_PATH=D:/noah/ChessWarrior/server/kbe/res/;D:/noah/ChessWarrior/server/chess_assets/;D:/noah/ChessWarrior/server/chess_assets/scripts/;D:/noah/ChessWarrior/server/chess_assets/res/
  INFO root [2018-05-21 15:50:21,946] - Resmgr::initialize: KBE_BIN_PATH=D:/noah/ChessWarrior/server/kbe/bin/server/
  INFO root [2018-05-21 15:50:21,951] - NetworkInterface::initialize(EXTERNAL): bound to all interfaces with default route interface on 192.168.0.111 ( 192.168.0.111:20013 )
  INFO root [2018-05-21 15:50:21,951] - NetworkInterface::initialize(EXTERNAL): address 192.168.0.111:20013, SOMAXCONN=128.
  INFO root [2018-05-21 15:50:21,952] - NetworkInterface::initialize(INTERNAL): bound to all interfaces with default route interface on 192.168.0.111 ( 192.168.0.111:56359 )
  INFO root [2018-05-21 15:50:21,952] - NetworkInterface::initialize(INTERNAL): address 192.168.0.111:56359, SOMAXCONN=128.
  INFO root [2018-05-21 15:50:21,952] - server-configs:
  INFO root [2018-05-21 15:50:21,952] - 	internalAddr : 192.168.0.111:56359
  INFO root [2018-05-21 15:50:21,952] - 	externalAddr : 192.168.0.111:20013
  INFO root [2018-05-21 15:50:21,952] - 	componentID : 9000
  INFO root [2018-05-21 15:50:23,642] - ---- loginapp Version: 2.0.0. ScriptVersion: 0.1.0. Protocol: B20F257E332779D9E1EA7D34638E6146. Config: Debug X86. Built: 15:50:13 May 15 2018. AppID: 9000. UID: 1611. PID: 1328 ----
  INFO root [2018-05-21 15:50:23,642] - Server : Intel(R) Core(TM) i5-7500 CPU @ 3.40GHz with 2097151kB RAM
  INFO root [2018-05-21 15:50:23,642] - ThreadPool::createThreadPool: creating  threadpool...
  INFO root [2018-05-21 15:50:23,642] - ThreadPool::createThreadPool: successfully(2), newThreadCount=1, normalMaxThreadCount=2, maxThreadCount=8
 DEBUG root [2018-05-21 15:50:23,779] - Script::install_py_dlls(): PyInit_pyexpat
 DEBUG root [2018-05-21 15:50:23,779] - Script::install_py_dlls(): PyInit__socket
 DEBUG root [2018-05-21 15:50:23,779] - Script::install_py_dlls(): PyInit__ssl
 DEBUG root [2018-05-21 15:50:23,783] - Script::install_py_dlls(): PyInit__hashlib
 DEBUG root [2018-05-21 15:50:23,784] - Script::install_py_dlls(): PyInit_select
 DEBUG root [2018-05-21 15:50:23,784] - Script::install_py_dlls(): PyInit__ctypes
 DEBUG root [2018-05-21 15:50:23,785] - Script::install_py_dlls(): PyInit__elementtree
 DEBUG root [2018-05-21 15:50:23,828] - Script::install_py_dlls(): PyInit_unicodedata
  INFO root [2018-05-21 15:50:23,871] - Script::install(): is successfully, Python=(3.4.2 (default, May 15 2018, 15:50:12) [MSC v.1900 32 bit (Intel)])!
  INFO root [2018-05-21 15:50:23,871] - Script::install(): KBExtra is successfully!
S_INFO root [2018-05-21 15:50:23,873] - onLoginAppReady: bootstrapGroupIndex=0, bootstrapGlobalIndex=0
  INFO root [2018-05-21 15:50:23,873] - TelnetServer server is running on port 31000
  INFO root [2018-05-21 15:50:24,388] - ---- loginapp is running ----
 DEBUG root [2018-05-21 15:50:24,389] - Components::process(): Request for the process of identity...
  WARN root [2018-05-21 15:50:27,877] - BundleBroadcast::BundleBroadcast: Cannot bind to port 20370, errorno=10048
  INFO root [2018-05-21 15:50:31,933] - Components::findComponents: find logger(1)...
  INFO root [2018-05-21 15:50:31,946] - Components::findComponents: found logger, addr:192.168.0.111:56362
  INFO root [2018-05-21 15:50:31,946] - Components::addComponent: reset orderLog, uid=1611!
  INFO root [2018-05-21 15:50:31,947] - Components::addComponent[logger], uid=1611, componentID=2000, globalorderid=1, grouporderid=1, totalcount=1
  INFO root [2018-05-21 15:50:31,948] - Components::findComponents: find dbmgr(2)...
  INFO root [2018-05-21 15:50:32,051] - Components::findComponents: found dbmgr, addr:192.168.0.111:56360
  INFO root [2018-05-21 15:50:32,051] - Components::addComponent: reset orderLog, uid=1611!
  INFO root [2018-05-21 15:50:32,051] - Components::addComponent[dbmgr], uid=1611, componentID=4000, globalorderid=1, grouporderid=1, totalcount=1
  INFO root [2018-05-21 15:50:32,052] - Components::findComponents: found dbmgr, addr:192.168.0.111:56360
  WARN root [2018-05-21 15:50:33,052] - Forwarding logs to logger[192.168.0.111:56362/0/0/0]...
 ERROR root [2018-05-21 15:51:26,339] - Components::removeComponentByChannel: logger : 2000, Abnormal exit! errorno=10054
  INFO root [2018-05-21 15:51:26,345] - The following logs sent to logger failed:
  WARN root [2018-05-21 15:51:26,345] - ==> [2018-05-21 15:51:26 329] TCPPacketReceiver::processPendingEvents(192.168.0.111:56362): Throwing REASON_GENERAL_NETWORK - WSAECONNRESET
 ERROR root [2018-05-21 15:51:26,345] - ==> [2018-05-21 15:51:26 339] Components::removeComponentByChannel: logger : 2000, Abnormal exit! errorno=10054
  WARN root [2018-05-21 15:51:26,370] - TCPPacketReceiver::processPendingEvents(192.168.0.111:56360): Throwing REASON_GENERAL_NETWORK - WSAECONNRESET
 ERROR root [2018-05-21 15:51:26,379] - Components::removeComponentByChannel: dbmgr : 4000, Abnormal exit! errorno=10054
  INFO root [2018-05-21 15:51:26,390] - Shutdowner::onShutdownBegin: shutting down(period=0, tickPeriod=1)
S_INFO root [2018-05-21 15:51:26,391] - onLoginAppShutDown()
  INFO root [2018-05-21 15:51:26,391] - Shutdowner::onShutdown: shutting down
  INFO root [2018-05-21 15:51:26,391] - Shutdowner::onShutdownEnd: shutting down
 DEBUG root [2018-05-21 15:51:26,391] - Script::uninstall_py_dlls(): PyInit_pyexpat
 DEBUG root [2018-05-21 15:51:26,391] - Script::uninstall_py_dlls(): PyInit__socket
 DEBUG root [2018-05-21 15:51:26,392] - Script::uninstall_py_dlls(): PyInit__ssl
 DEBUG root [2018-05-21 15:51:26,392] - Script::uninstall_py_dlls(): PyInit__hashlib
 DEBUG root [2018-05-21 15:51:26,392] - Script::uninstall_py_dlls(): PyInit_select
 DEBUG root [2018-05-21 15:51:26,392] - Script::uninstall_py_dlls(): PyInit__ctypes
 DEBUG root [2018-05-21 15:51:26,392] - Script::uninstall_py_dlls(): PyInit__elementtree
 DEBUG root [2018-05-21 15:51:26,392] - Script::uninstall_py_dlls(): PyInit_unicodedata
  INFO root [2018-05-21 15:51:26,403] - Script::uninstall(): is successfully!
 DEBUG root [2018-05-21 15:51:26,404] - ThreadPool::destroy(): starting size 2.
  WARN root [2018-05-21 15:51:26,704] - ThreadPool::destroy(): waiting for thread(2)[0x459c5b8,0x459c4e8,], try=1
  INFO root [2018-05-21 15:51:30,536] - 
  INFO root [2018-05-21 15:51:30,536] - -----------------------------------------------------------------------------------------


  INFO root [2018-05-21 15:51:30,537] - Resmgr::initialize: KBE_ROOT=D:/noah/ChessWarrior/server/
  INFO root [2018-05-21 15:51:30,537] - Resmgr::initialize: KBE_RES_PATH=D:/noah/ChessWarrior/server/kbe/res/;D:/noah/ChessWarrior/server/chess_assets/;D:/noah/ChessWarrior/server/chess_assets/scripts/;D:/noah/ChessWarrior/server/chess_assets/res/
  INFO root [2018-05-21 15:51:30,537] - Resmgr::initialize: KBE_BIN_PATH=D:/noah/ChessWarrior/server/kbe/bin/server/
  INFO root [2018-05-21 15:51:30,539] - NetworkInterface::initialize(EXTERNAL): bound to all interfaces with default route interface on 192.168.0.111 ( 192.168.0.111:20013 )
  INFO root [2018-05-21 15:51:30,539] - NetworkInterface::initialize(EXTERNAL): address 192.168.0.111:20013, SOMAXCONN=128.
  INFO root [2018-05-21 15:51:30,540] - NetworkInterface::initialize(INTERNAL): bound to all interfaces with default route interface on 192.168.0.111 ( 192.168.0.111:56515 )
  INFO root [2018-05-21 15:51:30,540] - NetworkInterface::initialize(INTERNAL): address 192.168.0.111:56515, SOMAXCONN=128.
  INFO root [2018-05-21 15:51:30,540] - server-configs:
  INFO root [2018-05-21 15:51:30,540] - 	internalAddr : 192.168.0.111:56515
  INFO root [2018-05-21 15:51:30,540] - 	externalAddr : 192.168.0.111:20013
  INFO root [2018-05-21 15:51:30,540] - 	componentID : 9000
  INFO root [2018-05-21 15:51:32,194] - ---- loginapp Version: 2.0.0. ScriptVersion: 0.1.0. Protocol: B20F257E332779D9E1EA7D34638E6146. Config: Debug X86. Built: 15:50:13 May 15 2018. AppID: 9000. UID: 1833. PID: 19328 ----
  INFO root [2018-05-21 15:51:32,194] - Server : Intel(R) Core(TM) i5-7500 CPU @ 3.40GHz with 2097151kB RAM
  INFO root [2018-05-21 15:51:32,195] - ThreadPool::createThreadPool: creating  threadpool...
  INFO root [2018-05-21 15:51:32,195] - ThreadPool::createThreadPool: successfully(2), newThreadCount=1, normalMaxThreadCount=2, maxThreadCount=8
 DEBUG root [2018-05-21 15:51:32,327] - Script::install_py_dlls(): PyInit_pyexpat
 DEBUG root [2018-05-21 15:51:32,327] - Script::install_py_dlls(): PyInit__socket
 DEBUG root [2018-05-21 15:51:32,327] - Script::install_py_dlls(): PyInit__ssl
 DEBUG root [2018-05-21 15:51:32,330] - Script::install_py_dlls(): PyInit__hashlib
 DEBUG root [2018-05-21 15:51:32,331] - Script::install_py_dlls(): PyInit_select
 DEBUG root [2018-05-21 15:51:32,331] - Script::install_py_dlls(): PyInit__ctypes
 DEBUG root [2018-05-21 15:51:32,332] - Script::install_py_dlls(): PyInit__elementtree
 DEBUG root [2018-05-21 15:51:32,378] - Script::install_py_dlls(): PyInit_unicodedata
  INFO root [2018-05-21 15:51:32,422] - Script::install(): is successfully, Python=(3.4.2 (default, May 15 2018, 15:50:12) [MSC v.1900 32 bit (Intel)])!
  INFO root [2018-05-21 15:51:32,422] - Script::install(): KBExtra is successfully!
S_INFO root [2018-05-21 15:51:32,425] - onLoginAppReady: bootstrapGroupIndex=0, bootstrapGlobalIndex=0
  INFO root [2018-05-21 15:51:32,425] - TelnetServer server is running on port 31000
  INFO root [2018-05-21 15:51:33,018] - ---- loginapp is running ----
 DEBUG root [2018-05-21 15:51:33,020] - Components::process(): Request for the process of identity...
  INFO root [2018-05-21 15:51:40,225] - Components::findComponents: find logger(1)...
  INFO root [2018-05-21 15:51:40,263] - Components::findComponents: found logger, addr:192.168.0.111:56518
  INFO root [2018-05-21 15:51:40,263] - Components::addComponent: reset orderLog, uid=1833!
  INFO root [2018-05-21 15:51:40,264] - Components::addComponent[logger], uid=1833, componentID=2000, globalorderid=1, grouporderid=1, totalcount=1
  INFO root [2018-05-21 15:51:40,265] - Components::findComponents: find dbmgr(2)...
  INFO root [2018-05-21 15:51:40,369] - Components::findComponents: found dbmgr, addr:192.168.0.111:56516
  INFO root [2018-05-21 15:51:40,369] - Components::addComponent: reset orderLog, uid=1833!
  INFO root [2018-05-21 15:51:40,369] - Components::addComponent[dbmgr], uid=1833, componentID=4000, globalorderid=1, grouporderid=1, totalcount=1
  INFO root [2018-05-21 15:51:40,370] - Components::findComponents: found dbmgr, addr:192.168.0.111:56516
  WARN root [2018-05-21 15:51:41,370] - Forwarding logs to logger[192.168.0.111:56518/0/0/0]...
 ERROR root [2018-05-21 15:52:11,978] - Components::removeComponentByChannel: logger : 2000, Abnormal exit! errorno=10054
  INFO root [2018-05-21 15:52:11,989] - The following logs sent to logger failed:
  WARN root [2018-05-21 15:52:11,989] - ==> [2018-05-21 15:52:11 973] TCPPacketReceiver::processPendingEvents(192.168.0.111:56518): Throwing REASON_GENERAL_NETWORK - WSAECONNRESET
 ERROR root [2018-05-21 15:52:11,989] - ==> [2018-05-21 15:52:11 979] Components::removeComponentByChannel: logger : 2000, Abnormal exit! errorno=10054
  WARN root [2018-05-21 15:52:12,021] - TCPPacketReceiver::processPendingEvents(192.168.0.111:56516): Throwing REASON_GENERAL_NETWORK - WSAECONNRESET
 ERROR root [2018-05-21 15:52:12,034] - Components::removeComponentByChannel: dbmgr : 4000, Abnormal exit! errorno=10054
  INFO root [2018-05-21 15:52:12,038] - Shutdowner::onShutdownBegin: shutting down(period=0, tickPeriod=1)
S_INFO root [2018-05-21 15:52:12,039] - onLoginAppShutDown()
  INFO root [2018-05-21 15:52:12,039] - Shutdowner::onShutdown: shutting down
  INFO root [2018-05-21 15:52:12,039] - Shutdowner::onShutdownEnd: shutting down
 DEBUG root [2018-05-21 15:52:12,039] - Script::uninstall_py_dlls(): PyInit_pyexpat
 DEBUG root [2018-05-21 15:52:12,039] - Script::uninstall_py_dlls(): PyInit__socket
 DEBUG root [2018-05-21 15:52:12,039] - Script::uninstall_py_dlls(): PyInit__ssl
 DEBUG root [2018-05-21 15:52:12,040] - Script::uninstall_py_dlls(): PyInit__hashlib
 DEBUG root [2018-05-21 15:52:12,040] - Script::uninstall_py_dlls(): PyInit_select
 DEBUG root [2018-05-21 15:52:12,040] - Script::uninstall_py_dlls(): PyInit__ctypes
 DEBUG root [2018-05-21 15:52:12,040] - Script::uninstall_py_dlls(): PyInit__elementtree
 DEBUG root [2018-05-21 15:52:12,040] - Script::uninstall_py_dlls(): PyInit_unicodedata
  INFO root [2018-05-21 15:52:12,054] - Script::uninstall(): is successfully!
 DEBUG root [2018-05-21 15:52:12,055] - ThreadPool::destroy(): starting size 2.
  WARN root [2018-05-21 15:52:12,355] - ThreadPool::destroy(): waiting for thread(2)[0x4536258,0x45362c0,], try=1
  INFO root [2018-05-21 15:52:15,526] - 
  INFO root [2018-05-21 15:52:15,533] - -----------------------------------------------------------------------------------------


  INFO root [2018-05-21 15:52:15,534] - Resmgr::initialize: KBE_ROOT=D:/noah/ChessWarrior/server/
  INFO root [2018-05-21 15:52:15,534] - Resmgr::initialize: KBE_RES_PATH=D:/noah/ChessWarrior/server/kbe/res/;D:/noah/ChessWarrior/server/chess_assets/;D:/noah/ChessWarrior/server/chess_assets/scripts/;D:/noah/ChessWarrior/server/chess_assets/res/
  INFO root [2018-05-21 15:52:15,534] - Resmgr::initialize: KBE_BIN_PATH=D:/noah/ChessWarrior/server/kbe/bin/server/
  INFO root [2018-05-21 15:52:15,544] - NetworkInterface::initialize(EXTERNAL): bound to all interfaces with default route interface on 192.168.0.111 ( 192.168.0.111:20013 )
  INFO root [2018-05-21 15:52:15,545] - NetworkInterface::initialize(EXTERNAL): address 192.168.0.111:20013, SOMAXCONN=128.
  INFO root [2018-05-21 15:52:15,545] - NetworkInterface::initialize(INTERNAL): bound to all interfaces with default route interface on 192.168.0.111 ( 192.168.0.111:56640 )
  INFO root [2018-05-21 15:52:15,545] - NetworkInterface::initialize(INTERNAL): address 192.168.0.111:56640, SOMAXCONN=128.
  INFO root [2018-05-21 15:52:15,546] - server-configs:
  INFO root [2018-05-21 15:52:15,546] - 	internalAddr : 192.168.0.111:56640
  INFO root [2018-05-21 15:52:15,546] - 	externalAddr : 192.168.0.111:20013
  INFO root [2018-05-21 15:52:15,546] - 	componentID : 9000
  INFO root [2018-05-21 15:52:17,208] - ---- loginapp Version: 2.0.0. ScriptVersion: 0.1.0. Protocol: B20F257E332779D9E1EA7D34638E6146. Config: Debug X86. Built: 15:50:13 May 15 2018. AppID: 9000. UID: 1980. PID: 2828 ----
  INFO root [2018-05-21 15:52:17,210] - Server : Intel(R) Core(TM) i5-7500 CPU @ 3.40GHz with 2097151kB RAM
  INFO root [2018-05-21 15:52:17,212] - ThreadPool::createThreadPool: creating  threadpool...
  INFO root [2018-05-21 15:52:17,213] - ThreadPool::createThreadPool: successfully(2), newThreadCount=1, normalMaxThreadCount=2, maxThreadCount=8
 DEBUG root [2018-05-21 15:52:17,392] - Script::install_py_dlls(): PyInit_pyexpat
 DEBUG root [2018-05-21 15:52:17,393] - Script::install_py_dlls(): PyInit__socket
 DEBUG root [2018-05-21 15:52:17,393] - Script::install_py_dlls(): PyInit__ssl
 DEBUG root [2018-05-21 15:52:17,396] - Script::install_py_dlls(): PyInit__hashlib
 DEBUG root [2018-05-21 15:52:17,397] - Script::install_py_dlls(): PyInit_select
 DEBUG root [2018-05-21 15:52:17,397] - Script::install_py_dlls(): PyInit__ctypes
 DEBUG root [2018-05-21 15:52:17,397] - Script::install_py_dlls(): PyInit__elementtree
 DEBUG root [2018-05-21 15:52:17,459] - Script::install_py_dlls(): PyInit_unicodedata
  INFO root [2018-05-21 15:52:17,556] - Script::install(): is successfully, Python=(3.4.2 (default, May 15 2018, 15:50:12) [MSC v.1900 32 bit (Intel)])!
  INFO root [2018-05-21 15:52:17,556] - Script::install(): KBExtra is successfully!
S_INFO root [2018-05-21 15:52:17,560] - onLoginAppReady: bootstrapGroupIndex=0, bootstrapGlobalIndex=0
  INFO root [2018-05-21 15:52:17,561] - TelnetServer server is running on port 31000
  INFO root [2018-05-21 15:52:18,183] - ---- loginapp is running ----
 DEBUG root [2018-05-21 15:52:18,210] - Components::process(): Request for the process of identity...
  INFO root [2018-05-21 15:52:25,500] - Components::findComponents: find logger(1)...
  INFO root [2018-05-21 15:52:25,530] - Components::findComponents: found logger, addr:192.168.0.111:56637
  INFO root [2018-05-21 15:52:25,530] - Components::addComponent: reset orderLog, uid=1980!
  INFO root [2018-05-21 15:52:25,531] - Components::addComponent[logger], uid=1980, componentID=2000, globalorderid=1, grouporderid=1, totalcount=1
  INFO root [2018-05-21 15:52:25,532] - Components::findComponents: find dbmgr(2)...
  INFO root [2018-05-21 15:52:25,636] - Components::findComponents: found dbmgr, addr:192.168.0.111:56638
  INFO root [2018-05-21 15:52:25,637] - Components::addComponent: reset orderLog, uid=1980!
  INFO root [2018-05-21 15:52:25,637] - Components::addComponent[dbmgr], uid=1980, componentID=4000, globalorderid=1, grouporderid=1, totalcount=1
  INFO root [2018-05-21 15:52:25,637] - Components::findComponents: found dbmgr, addr:192.168.0.111:56638
  WARN root [2018-05-21 15:52:26,638] - Forwarding logs to logger[192.168.0.111:56637/0/0/0]...
 ERROR root [2018-05-21 15:53:04,834] - Components::removeComponentByChannel: logger : 2000, Abnormal exit! errorno=10054
  INFO root [2018-05-21 15:53:04,849] - The following logs sent to logger failed:
  WARN root [2018-05-21 15:53:04,849] - ==> [2018-05-21 15:53:04 833] TCPPacketReceiver::processPendingEvents(192.168.0.111:56637): Throwing REASON_GENERAL_NETWORK - WSAECONNRESET
 ERROR root [2018-05-21 15:53:04,850] - ==> [2018-05-21 15:53:04 834] Components::removeComponentByChannel: logger : 2000, Abnormal exit! errorno=10054
  WARN root [2018-05-21 15:53:04,908] - TCPPacketReceiver::processPendingEvents(192.168.0.111:56638): Throwing REASON_GENERAL_NETWORK - WSAECONNRESET
 ERROR root [2018-05-21 15:53:04,915] - Components::removeComponentByChannel: dbmgr : 4000, Abnormal exit! errorno=10054
  INFO root [2018-05-21 15:53:04,920] - Shutdowner::onShutdownBegin: shutting down(period=0, tickPeriod=1)
S_INFO root [2018-05-21 15:53:04,921] - onLoginAppShutDown()
  INFO root [2018-05-21 15:53:04,921] - Shutdowner::onShutdown: shutting down
  INFO root [2018-05-21 15:53:04,921] - Shutdowner::onShutdownEnd: shutting down
 DEBUG root [2018-05-21 15:53:04,922] - Script::uninstall_py_dlls(): PyInit_pyexpat
 DEBUG root [2018-05-21 15:53:04,922] - Script::uninstall_py_dlls(): PyInit__socket
 DEBUG root [2018-05-21 15:53:04,922] - Script::uninstall_py_dlls(): PyInit__ssl
 DEBUG root [2018-05-21 15:53:04,922] - Script::uninstall_py_dlls(): PyInit__hashlib
 DEBUG root [2018-05-21 15:53:04,922] - Script::uninstall_py_dlls(): PyInit_select
 DEBUG root [2018-05-21 15:53:04,922] - Script::uninstall_py_dlls(): PyInit__ctypes
 DEBUG root [2018-05-21 15:53:04,922] - Script::uninstall_py_dlls(): PyInit__elementtree
 DEBUG root [2018-05-21 15:53:04,922] - Script::uninstall_py_dlls(): PyInit_unicodedata
  INFO root [2018-05-21 15:53:04,933] - Script::uninstall(): is successfully!
 DEBUG root [2018-05-21 15:53:04,933] - ThreadPool::destroy(): starting size 2.
  WARN root [2018-05-21 15:53:05,234] - ThreadPool::destroy(): waiting for thread(2)[0xdd0680,0xdd0b60,], try=1
  INFO root [2018-05-21 15:54:35,984] - 
  INFO root [2018-05-21 15:54:36,000] - -----------------------------------------------------------------------------------------


  INFO root [2018-05-21 15:54:36,001] - Resmgr::initialize: KBE_ROOT=D:/noah/ChessWarrior/server/
  INFO root [2018-05-21 15:54:36,001] - Resmgr::initialize: KBE_RES_PATH=D:/noah/ChessWarrior/server/kbe/res/;D:/noah/ChessWarrior/server/chess_assets/;D:/noah/ChessWarrior/server/chess_assets/scripts/;D:/noah/ChessWarrior/server/chess_assets/res/
  INFO root [2018-05-21 15:54:36,001] - Resmgr::initialize: KBE_BIN_PATH=D:/noah/ChessWarrior/server/kbe/bin/server/
  INFO root [2018-05-21 15:54:36,018] - NetworkInterface::initialize(EXTERNAL): bound to all interfaces with default route interface on 192.168.0.111 ( 192.168.0.111:20013 )
  INFO root [2018-05-21 15:54:36,019] - NetworkInterface::initialize(EXTERNAL): address 192.168.0.111:20013, SOMAXCONN=128.
  INFO root [2018-05-21 15:54:36,019] - NetworkInterface::initialize(INTERNAL): bound to all interfaces with default route interface on 192.168.0.111 ( 192.168.0.111:56892 )
  INFO root [2018-05-21 15:54:36,019] - NetworkInterface::initialize(INTERNAL): address 192.168.0.111:56892, SOMAXCONN=128.
  INFO root [2018-05-21 15:54:36,020] - server-configs:
  INFO root [2018-05-21 15:54:36,020] - 	internalAddr : 192.168.0.111:56892
  INFO root [2018-05-21 15:54:36,020] - 	externalAddr : 192.168.0.111:20013
  INFO root [2018-05-21 15:54:36,020] - 	componentID : 9000
  INFO root [2018-05-21 15:54:36,335] - Components::findLogger: found logger, addr:192.168.0.111:56888
  INFO root [2018-05-21 15:54:36,335] - Components::addComponent: reset orderLog, uid=2437!
  INFO root [2018-05-21 15:54:36,335] - Components::addComponent[logger], uid=2437, componentID=2000, globalorderid=1, grouporderid=1, totalcount=1
  INFO root [2018-05-21 15:54:36,343] - ---- loginapp Version: 2.0.0. ScriptVersion: 0.1.0. Protocol: B20F257E332779D9E1EA7D34638E6146. Config: Debug X86. Built: 15:50:13 May 15 2018. AppID: 9000. UID: 2437. PID: 9912 ----
  INFO root [2018-05-21 15:54:36,347] - Server : Intel(R) Core(TM) i5-7500 CPU @ 3.40GHz with 2097151kB RAM
  INFO root [2018-05-21 15:54:36,348] - ThreadPool::createThreadPool: creating  threadpool...
  INFO root [2018-05-21 15:54:36,348] - ThreadPool::createThreadPool: successfully(2), newThreadCount=1, normalMaxThreadCount=2, maxThreadCount=8
 DEBUG root [2018-05-21 15:54:36,485] - Script::install_py_dlls(): PyInit_pyexpat
 DEBUG root [2018-05-21 15:54:36,486] - Script::install_py_dlls(): PyInit__socket
 DEBUG root [2018-05-21 15:54:36,486] - Script::install_py_dlls(): PyInit__ssl
 DEBUG root [2018-05-21 15:54:36,489] - Script::install_py_dlls(): PyInit__hashlib
 DEBUG root [2018-05-21 15:54:36,489] - Script::install_py_dlls(): PyInit_select
 DEBUG root [2018-05-21 15:54:36,489] - Script::install_py_dlls(): PyInit__ctypes
 DEBUG root [2018-05-21 15:54:36,490] - Script::install_py_dlls(): PyInit__elementtree
 DEBUG root [2018-05-21 15:54:36,532] - Script::install_py_dlls(): PyInit_unicodedata
  INFO root [2018-05-21 15:54:36,573] - Script::install(): is successfully, Python=(3.4.2 (default, May 15 2018, 15:50:12) [MSC v.1900 32 bit (Intel)])!
  INFO root [2018-05-21 15:54:36,573] - Script::install(): KBExtra is successfully!
S_INFO root [2018-05-21 15:54:36,577] - onLoginAppReady: bootstrapGroupIndex=0, bootstrapGlobalIndex=0
  INFO root [2018-05-21 15:54:36,577] - TelnetServer server is running on port 31000
  INFO root [2018-05-21 15:54:36,991] - ---- loginapp is running ----
 DEBUG root [2018-05-21 15:54:36,991] - Components::process(): Request for the process of identity...
  WARN root [2018-05-21 15:54:43,156] - Forwarding logs to logger[192.168.0.111:56888/0/0/0]...
 ERROR root [2018-05-21 15:55:35,596] - Components::removeComponentByChannel: logger : 2000, Abnormal exit! errorno=10054
  INFO root [2018-05-21 15:55:35,601] - The following logs sent to logger failed:
  WARN root [2018-05-21 15:55:35,601] - ==> [2018-05-21 15:55:35 591] TCPPacketReceiver::processPendingEvents(192.168.0.111:56888): Throwing REASON_GENERAL_NETWORK - WSAECONNRESET
 ERROR root [2018-05-21 15:55:35,601] - ==> [2018-05-21 15:55:35 597] Components::removeComponentByChannel: logger : 2000, Abnormal exit! errorno=10054
  WARN root [2018-05-21 15:55:35,645] - TCPPacketReceiver::processPendingEvents(192.168.0.111:56889): Throwing REASON_GENERAL_NETWORK - WSAECONNRESET
 ERROR root [2018-05-21 15:55:35,648] - Components::removeComponentByChannel: dbmgr : 4000, Abnormal exit! errorno=10054
  INFO root [2018-05-21 15:55:35,662] - Shutdowner::onShutdownBegin: shutting down(period=0, tickPeriod=1)
S_INFO root [2018-05-21 15:55:35,663] - onLoginAppShutDown()
  INFO root [2018-05-21 15:55:35,663] - Shutdowner::onShutdown: shutting down
  INFO root [2018-05-21 15:55:35,664] - Shutdowner::onShutdownEnd: shutting down
 DEBUG root [2018-05-21 15:55:35,664] - Script::uninstall_py_dlls(): PyInit_pyexpat
 DEBUG root [2018-05-21 15:55:35,664] - Script::uninstall_py_dlls(): PyInit__socket
 DEBUG root [2018-05-21 15:55:35,664] - Script::uninstall_py_dlls(): PyInit__ssl
 DEBUG root [2018-05-21 15:55:35,665] - Script::uninstall_py_dlls(): PyInit__hashlib
 DEBUG root [2018-05-21 15:55:35,665] - Script::uninstall_py_dlls(): PyInit_select
 DEBUG root [2018-05-21 15:55:35,665] - Script::uninstall_py_dlls(): PyInit__ctypes
 DEBUG root [2018-05-21 15:55:35,665] - Script::uninstall_py_dlls(): PyInit__elementtree
 DEBUG root [2018-05-21 15:55:35,665] - Script::uninstall_py_dlls(): PyInit_unicodedata
  INFO root [2018-05-21 15:55:35,675] - Script::uninstall(): is successfully!
 DEBUG root [2018-05-21 15:55:35,675] - ThreadPool::destroy(): starting size 2.
  WARN root [2018-05-21 15:55:35,975] - ThreadPool::destroy(): waiting for thread(2)[0x47190b0,0x4719b40,], try=1
  INFO root [2018-05-21 15:57:36,500] - 
  INFO root [2018-05-21 15:57:36,500] - -----------------------------------------------------------------------------------------


  INFO root [2018-05-21 15:57:36,501] - Resmgr::initialize: KBE_ROOT=D:/noah/ChessWarrior/server/
  INFO root [2018-05-21 15:57:36,501] - Resmgr::initialize: KBE_RES_PATH=D:/noah/ChessWarrior/server/kbe/res/;D:/noah/ChessWarrior/server/chess_assets/;D:/noah/ChessWarrior/server/chess_assets/scripts/;D:/noah/ChessWarrior/server/chess_assets/res/
  INFO root [2018-05-21 15:57:36,501] - Resmgr::initialize: KBE_BIN_PATH=D:/noah/ChessWarrior/server/kbe/bin/server/
  INFO root [2018-05-21 15:57:36,503] - NetworkInterface::initialize(EXTERNAL): bound to all interfaces with default route interface on 192.168.0.111 ( 192.168.0.111:20013 )
  INFO root [2018-05-21 15:57:36,503] - NetworkInterface::initialize(EXTERNAL): address 192.168.0.111:20013, SOMAXCONN=128.
  INFO root [2018-05-21 15:57:36,504] - NetworkInterface::initialize(INTERNAL): bound to all interfaces with default route interface on 192.168.0.111 ( 192.168.0.111:57223 )
  INFO root [2018-05-21 15:57:36,504] - NetworkInterface::initialize(INTERNAL): address 192.168.0.111:57223, SOMAXCONN=128.
  INFO root [2018-05-21 15:57:36,504] - server-configs:
  INFO root [2018-05-21 15:57:36,504] - 	internalAddr : 192.168.0.111:57223
  INFO root [2018-05-21 15:57:36,505] - 	externalAddr : 192.168.0.111:20013
  INFO root [2018-05-21 15:57:36,505] - 	componentID : 9000
  INFO root [2018-05-21 15:57:38,214] - ---- loginapp Version: 2.0.0. ScriptVersion: 0.1.0. Protocol: B20F257E332779D9E1EA7D34638E6146. Config: Debug X86. Built: 15:50:13 May 15 2018. AppID: 9000. UID: 3028. PID: 8672 ----
  INFO root [2018-05-21 15:57:38,215] - Server : Intel(R) Core(TM) i5-7500 CPU @ 3.40GHz with 2097151kB RAM
  INFO root [2018-05-21 15:57:38,215] - ThreadPool::createThreadPool: creating  threadpool...
  INFO root [2018-05-21 15:57:38,215] - ThreadPool::createThreadPool: successfully(2), newThreadCount=1, normalMaxThreadCount=2, maxThreadCount=8
 DEBUG root [2018-05-21 15:57:38,408] - Script::install_py_dlls(): PyInit_pyexpat
 DEBUG root [2018-05-21 15:57:38,409] - Script::install_py_dlls(): PyInit__socket
 DEBUG root [2018-05-21 15:57:38,409] - Script::install_py_dlls(): PyInit__ssl
 DEBUG root [2018-05-21 15:57:38,412] - Script::install_py_dlls(): PyInit__hashlib
 DEBUG root [2018-05-21 15:57:38,413] - Script::install_py_dlls(): PyInit_select
 DEBUG root [2018-05-21 15:57:38,413] - Script::install_py_dlls(): PyInit__ctypes
 DEBUG root [2018-05-21 15:57:38,414] - Script::install_py_dlls(): PyInit__elementtree
 DEBUG root [2018-05-21 15:57:38,471] - Script::install_py_dlls(): PyInit_unicodedata
  INFO root [2018-05-21 15:57:38,514] - Script::install(): is successfully, Python=(3.4.2 (default, May 15 2018, 15:50:12) [MSC v.1900 32 bit (Intel)])!
  INFO root [2018-05-21 15:57:38,514] - Script::install(): KBExtra is successfully!
S_INFO root [2018-05-21 15:57:38,518] - onLoginAppReady: bootstrapGroupIndex=0, bootstrapGlobalIndex=0
  INFO root [2018-05-21 15:57:38,518] - TelnetServer server is running on port 31000
  INFO root [2018-05-21 15:57:39,128] - ---- loginapp is running ----
 DEBUG root [2018-05-21 15:57:39,130] - Components::process(): Request for the process of identity...
  INFO root [2018-05-21 15:57:46,339] - Components::findComponents: find logger(1)...
  INFO root [2018-05-21 15:57:46,347] - Components::findComponents: found logger, addr:192.168.0.111:57230
  INFO root [2018-05-21 15:57:46,347] - Components::addComponent: reset orderLog, uid=3028!
  INFO root [2018-05-21 15:57:46,347] - Components::addComponent[logger], uid=3028, componentID=2000, globalorderid=1, grouporderid=1, totalcount=1
  INFO root [2018-05-21 15:57:46,348] - Components::findComponents: find dbmgr(2)...
  INFO root [2018-05-21 15:57:46,451] - Components::findComponents: found dbmgr, addr:192.168.0.111:57231
  INFO root [2018-05-21 15:57:46,451] - Components::addComponent: reset orderLog, uid=3028!
  INFO root [2018-05-21 15:57:46,451] - Components::addComponent[dbmgr], uid=3028, componentID=4000, globalorderid=1, grouporderid=1, totalcount=1
  INFO root [2018-05-21 15:57:46,451] - Components::findComponents: found dbmgr, addr:192.168.0.111:57231
  WARN root [2018-05-21 15:57:47,451] - Forwarding logs to logger[192.168.0.111:57230/0/0/0]...
 ERROR root [2018-05-21 15:59:18,575] - Components::removeComponentByChannel: logger : 2000, Abnormal exit! errorno=10054
  INFO root [2018-05-21 15:59:18,576] - The following logs sent to logger failed:
  WARN root [2018-05-21 15:59:18,576] - ==> [2018-05-21 15:59:18 565] TCPPacketReceiver::processPendingEvents(192.168.0.111:57230): Throwing REASON_GENERAL_NETWORK - WSAECONNRESET
 ERROR root [2018-05-21 15:59:18,576] - ==> [2018-05-21 15:59:18 576] Components::removeComponentByChannel: logger : 2000, Abnormal exit! errorno=10054
  WARN root [2018-05-21 15:59:18,602] - TCPPacketReceiver::processPendingEvents(192.168.0.111:57231): Throwing REASON_GENERAL_NETWORK - WSAECONNRESET
 ERROR root [2018-05-21 15:59:18,608] - Components::removeComponentByChannel: dbmgr : 4000, Abnormal exit! errorno=10054
  INFO root [2018-05-21 15:59:18,611] - Shutdowner::onShutdownBegin: shutting down(period=0, tickPeriod=1)
S_INFO root [2018-05-21 15:59:18,613] - onLoginAppShutDown()
  INFO root [2018-05-21 15:59:18,613] - Shutdowner::onShutdown: shutting down
  INFO root [2018-05-21 15:59:18,613] - Shutdowner::onShutdownEnd: shutting down
 DEBUG root [2018-05-21 15:59:18,613] - Script::uninstall_py_dlls(): PyInit_pyexpat
 DEBUG root [2018-05-21 15:59:18,613] - Script::uninstall_py_dlls(): PyInit__socket
 DEBUG root [2018-05-21 15:59:18,613] - Script::uninstall_py_dlls(): PyInit__ssl
 DEBUG root [2018-05-21 15:59:18,613] - Script::uninstall_py_dlls(): PyInit__hashlib
 DEBUG root [2018-05-21 15:59:18,613] - Script::uninstall_py_dlls(): PyInit_select
 DEBUG root [2018-05-21 15:59:18,614] - Script::uninstall_py_dlls(): PyInit__ctypes
 DEBUG root [2018-05-21 15:59:18,614] - Script::uninstall_py_dlls(): PyInit__elementtree
 DEBUG root [2018-05-21 15:59:18,614] - Script::uninstall_py_dlls(): PyInit_unicodedata
  INFO root [2018-05-21 15:59:18,624] - Script::uninstall(): is successfully!
 DEBUG root [2018-05-21 15:59:18,624] - ThreadPool::destroy(): starting size 2.
  WARN root [2018-05-21 15:59:18,925] - ThreadPool::destroy(): waiting for thread(2)[0x96cba0,0x96cad0,], try=1
